:orphan:




XPEHH/CMS.GAS/3/1
=================

This page describes a signal of selection found in the
:doc:`../../../../../population/CMS` population
when compared with the :doc:`../../../../../population/GAS` population
using the :doc:`../../../../../method/XPEHH` statistic.The focus of this signal is on chromosome arm
**3R** between positions **43,920,000** and
**44,560,000**.
The evidence supporting this signal is
**moderate** (:math:`\Delta_{i}` >= 50 on both flanks).





.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----


The following 37 genes overlap the focal region: :doc:`../../../../../gene/AGAP009828`,  :doc:`../../../../../gene/AGAP009829` (beat protein),  :doc:`../../../../../gene/AGAP009830`,  :doc:`../../../../../gene/AGAP009831`,  :doc:`../../../../../gene/AGAP009832` (CASPS8 - short caspase 8),  :doc:`../../../../../gene/AGAP028572` (Gustatory receptor),  :doc:`../../../../../gene/AGAP009833` (voltage-dependent anion-selective channel protein 2),  :doc:`../../../../../gene/AGAP009834` (COP9 signalosome subunit 4),  :doc:`../../../../../gene/AGAP009835` (ABCC14 - ATP-binding cassette transporter (ABC transporter) family C member 14),  :doc:`../../../../../gene/AGAP009836`,  :doc:`../../../../../gene/AGAP009837`,  :doc:`../../../../../gene/AGAP009838` (Non-imprinted in Prader-Willi/Angelman syndrome region protein 2-like protein),  :doc:`../../../../../gene/AGAP009839` (Phosphatase 1 regulatory subunit 7),  :doc:`../../../../../gene/AGAP009840` (USO1 vesicle docking protein homolog),  :doc:`../../../../../gene/AGAP009841`,  :doc:`../../../../../gene/AGAP009842` (ribonuclease T2),  :doc:`../../../../../gene/AGAP009843`,  :doc:`../../../../../gene/AGAP009844` (CLIPB15 - CLIP-domain serine protease),  :doc:`../../../../../gene/AGAP009845`,  :doc:`../../../../../gene/AGAP009846` (Ras-related protein Rab-9A),  :doc:`../../../../../gene/AGAP009847` (Outspread),  :doc:`../../../../../gene/AGAP009848`,  :doc:`../../../../../gene/AGAP009849` (SP14D1),  :doc:`../../../../../gene/AGAP009850` (ABCG7 - ATP-binding cassette transporter (ABC transporter) family G member 7),  :doc:`../../../../../gene/AGAP009851`,  :doc:`../../../../../gene/AGAP009852` (mitochondrial fission 1 protein),  :doc:`../../../../../gene/AGAP009853`:sup:`3` (Gr5 - gustatory receptor 5),  :doc:`../../../../../gene/AGAP009854`:sup:`3` (Gr6 - gustatory receptor 6),  :doc:`../../../../../gene/AGAP009855`:sup:`3` (Gr7 - gustatory receptor 7),  :doc:`../../../../../gene/AGAP009856`:sup:`3` (Gr8 - gustatory receptor 8),  :doc:`../../../../../gene/AGAP009857`:sup:`3` (Gr4 - gustatory receptor 4),  :doc:`../../../../../gene/AGAP009858`:sup:`3` (Gr3 - gustatory receptor 3),  :doc:`../../../../../gene/AGAP009859`,  :doc:`../../../../../gene/AGAP009860`,  :doc:`../../../../../gene/AGAP009861` (Profilin protein),  :doc:`../../../../../gene/AGAP009862`,  :doc:`../../../../../gene/AGAP009863` (eukaryotic translation initiation factor 4A, isoform 2).



The following 8 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP009864`,  :doc:`../../../../../gene/AGAP009865`:sup:`1` (NADH dehydrogenase (ubiquinone) 1 beta subcomplex 9),  :doc:`../../../../../gene/AGAP009866` (dolichol-phosphate mannosyltransferase),  :doc:`../../../../../gene/AGAP009867` (AFG1-family ATPase),  :doc:`../../../../../gene/AGAP009868`:sup:`4` (CPR73 - cuticular protein RR-1 family 73),  :doc:`../../../../../gene/AGAP009869`:sup:`4` (CPR74 - cuticular protein RR-1 family 74),  :doc:`../../../../../gene/AGAP009870`:sup:`4` (CPR151 - cuticular protein RR-2 family 151),  :doc:`../../../../../gene/AGAP009871`:sup:`4` (CPR75 - cuticular protein RR-1 family 75).


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping selection signals
-----------------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak model :math:`\Delta_{i}`
      - Max. percentile
      - Known locus
    * - :doc:`../../../../../signal/XPEHH/AOM.GAS/3/1/index`
      - XPEHH
      - Angola *An. coluzzii*
      - 3R:43,460,000-44,120,000
      - 605
      - 99.9%
      - nan
    * - :doc:`../../../../../signal/XPEHH/AOM.BFM/3/1/index`
      - XPEHH
      - Angola *An. coluzzii*
      - 3R:43,860,000-44,500,000
      - 314
      - 99.7%
      - nan
    * - :doc:`../../../../../signal/H12/UGS/3/3/index`
      - H12
      - Uganda *An. gambiae*
      - 3R:44,380,000-45,140,000
      - 292
      - 95.0%
      - nan
    * - :doc:`../../../../../signal/XPEHH/UGS.CMS/3/2/index`
      - XPEHH
      - Uganda *An. gambiae*
      - 3R:43,940,000-43,980,000
      - 263
      - 99.7%
      - nan
    * - :doc:`../../../../../signal/XPEHH/AOM.GWA/3/1/index`
      - XPEHH
      - Angola *An. coluzzii*
      - 3R:44,320,000-44,480,000
      - 253
      - 99.3%
      - nan
    * - :doc:`../../../../../signal/XPEHH/UGS.BFS/3/2/index`
      - XPEHH
      - Uganda *An. gambiae*
      - 3R:43,920,000-43,980,000
      - 214
      - 99.7%
      - nan
    * - :doc:`../../../../../signal/IHS/UGS/3/3/index`
      - IHS
      - Uganda *An. gambiae*
      - 3R:43,800,000-43,920,000
      - 160
      - 97.7%
      - nan
    * - :doc:`../../../../../signal/IHS/GAS/3/1/index`
      - IHS
      - Gabon *An. gambiae*
      - 3R:44,140,000-44,300,000
      - 133
      - 95.5%
      - nan
    * - :doc:`../../../../../signal/IHS/AOM/3/1/index`
      - IHS
      - Angola *An. coluzzii*
      - 3R:43,740,000-44,560,000
      - 112
      - 99.2%
      - nan
    * - :doc:`../../../../../signal/XPEHH/UGS.BFS/3/6/index`
      - XPEHH
      - Uganda *An. gambiae*
      - 3R:44,440,000-44,520,000
      - 95
      - 99.7%
      - nan
    * - :doc:`../../../../../signal/XPEHH/UGS.GWA/3/3/index`
      - XPEHH
      - Uganda *An. gambiae*
      - 3R:43,940,000-44,000,000
      - 93
      - 99.7%
      - nan
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GAS/3/1/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GAS/3/1/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GAS/3/1/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_gaussian)
    [[Fit Statistics]]
        # function evals   = 63
        # data points      = 642
        # variables        = 4
        chi-square         = 75.120
        reduced chi-square = 0.118
        Akaike info crit   = -1369.411
        Bayesian info crit = -1351.553
    [[Variables]]
        center:      0 (fixed)
        amplitude:   1.06040037 +/- 0.037537 (3.54%) (init= 3)
        sigma:       2.99999993 +/- 0.149758 (4.99%) (init= 0.5)
        skew:       -0.46105339 +/- 0.043519 (9.44%) (init= 0)
        baseline:    1.20826728 +/- 0.031213 (2.58%) (init= 1)
        ceiling:     100 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(sigma, baseline)           = -0.712 
        C(amplitude, baseline)       = -0.697 
        C(amplitude, sigma)          =  0.226 
        C(skew, baseline)            = -0.198 
        C(amplitude, skew)           =  0.183 
        C(sigma, skew)               =  0.110 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 11
        # data points      = 641
        # variables        = 1
        chi-square         = 174.745
        reduced chi-square = 0.273
        Akaike info crit   = -831.108
        Bayesian info crit = -826.645
    [[Variables]]
        c:   1.69987682 +/- 0.020638 (1.21%) (init= 1)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


