<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <title>Burkina Faso An. gambiae | H12 | Chromosome 2 | Signal #2 &#8212; Ag1000G Selection Atlas</title>
    <link rel="stylesheet" href="../../../../../_static/bootstrap-sphinx.css" type="text/css" />
    <link rel="stylesheet" href="../../../../../_static/pygments.css" type="text/css" />
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    '../../../../../',
        VERSION:     'DEVELOPMENT VERSION',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true,
        SOURCELINK_SUFFIX: '.txt'
      };
    </script>
    <script type="text/javascript" src="../../../../../_static/jquery.js"></script>
    <script type="text/javascript" src="../../../../../_static/underscore.js"></script>
    <script type="text/javascript" src="../../../../../_static/doctools.js"></script>
    <script type="text/javascript" src="../../../../../_static/js/jquery-1.11.0.min.js"></script>
    <script type="text/javascript" src="../../../../../_static/js/jquery-fix.js"></script>
    <script type="text/javascript" src="../../../../../_static/bootstrap-3.3.7/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="../../../../../_static/bootstrap-sphinx.js"></script>
    <link rel="index" title="Index" href="../../../../../genindex.html" />
    <link rel="search" title="Search" href="../../../../../search.html" />
<meta charset='utf-8'>
<meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
<meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1'>
<meta name="apple-mobile-web-app-capable" content="yes">

  </head>
  <body>

  <div id="navbar" class="navbar navbar-default navbar-fixed-top">
    <div class="container">
      <div class="navbar-header">
        <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".nav-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a class="navbar-brand" href="../../../../../index.html">
          Ag1000G Selection Atlas</a>
        <span class="navbar-text navbar-version pull-left"><b>DEVELOPMENT VERSION</b></span>
      </div>

        <div class="collapse navbar-collapse nav-collapse">
          <ul class="nav navbar-nav">
            
            
              <li class="dropdown globaltoc-container">
  <a role="button"
     id="dLabelGlobalToc"
     data-toggle="dropdown"
     data-target="#"
     href="../../../../../index.html">Site <b class="caret"></b></a>
  <ul class="dropdown-menu globaltoc"
      role="menu"
      aria-labelledby="dLabelGlobalToc"><ul>
<li class="toctree-l1"><a class="reference internal" href="../../../../../signals.html">All signals</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../../populations.html">Signals by population</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../../methods.html">Methods</a></li>
</ul>
</ul>
</li>
              
                <li class="dropdown">
  <a role="button"
     id="dLabelLocalToc"
     data-toggle="dropdown"
     data-target="#"
     href="#">Page <b class="caret"></b></a>
  <ul class="dropdown-menu localtoc"
      role="menu"
      aria-labelledby="dLabelLocalToc"><ul>
<li><a class="reference internal" href="#">Burkina Faso <em>An. gambiae</em> | H12 | Chromosome 2 | Signal #2</a><ul>
<li><a class="reference internal" href="#overlapping-signals">Overlapping signals</a></li>
<li><a class="reference internal" href="#diagnostics">Diagnostics</a><ul>
<li><a class="reference internal" href="#model-fit-reports">Model fit reports</a></li>
</ul>
</li>
<li><a class="reference internal" href="#comments">Comments</a></li>
</ul>
</li>
</ul>
</ul>
</li>
              
            
            
              
                
              
            
            
            
            
          </ul>

          
            
<form class="navbar-form navbar-right" action="../../../../../search.html" method="get">
 <div class="form-group">
  <input type="text" name="q" class="form-control" placeholder="Search" />
 </div>
  <input type="hidden" name="check_keywords" value="yes" />
  <input type="hidden" name="area" value="default" />
</form>
          
        </div>
    </div>
  </div>

<div class="container">
  <div class="row">
    <div class="col-md-12 content">
      
  <div class="section" id="burkina-faso-an-gambiae-h12-chromosome-2-signal-2">
<h1>Burkina Faso <em>An. gambiae</em> | H12 | Chromosome 2 | Signal #2<a class="headerlink" href="#burkina-faso-an-gambiae-h12-chromosome-2-signal-2" title="Permalink to this headline">Â¶</a></h1>
<p>This page describes a signal of selection found in the
<a class="reference internal" href="../../../../../population/BFS.html"><span class="doc">Burkina Faso An. gambiae</span></a> population using the
<a class="reference internal" href="../../../../../method/H12.html"><span class="doc">H12 (Garud et al. 20XX)</span></a> statistic.The inferred focus of this signal is on chromosome arm
<strong>2L</strong> between positions <strong>28,500,001</strong> and
<strong>28,620,000</strong>.</p>
<p>The following 34 genes overlap the focal region: <a class="reference internal" href="../../../../../gene/AGAP006219.html"><span class="doc">AGAP006219</span></a> (receptor tyrosine kinase-like orphan receptor 1),  <a class="reference internal" href="../../../../../gene/AGAP006220.html"><span class="doc">AGAP006220</span></a> (aldehyde oxidase),  <a class="reference internal" href="../../../../../gene/AGAP006221.html"><span class="doc">AGAP006221</span></a> (aldehyde oxidase),  <a class="reference internal" href="../../../../../gene/AGAP006222.html"><span class="doc">AGAP006222</span></a> (glucosyl/glucuronosyl transferases),  <a class="reference internal" href="../../../../../gene/AGAP006223.html"><span class="doc">AGAP006223</span></a> (glucosyl/glucuronosyl transferases),  <a class="reference internal" href="../../../../../gene/AGAP006224.html"><span class="doc">AGAP006224</span></a> (aldehyde oxidase),  <a class="reference internal" href="../../../../../gene/AGAP006225.html"><span class="doc">AGAP006225</span></a> (aldehyde oxidase),  <a class="reference internal" href="../../../../../gene/AGAP006226.html"><span class="doc">AGAP006226</span></a> (Aldehyde_oxidase),  <a class="reference internal" href="../../../../../gene/AGAP006227.html"><span class="doc">AGAP006227</span></a> (alpha esterase),  <a class="reference internal" href="../../../../../gene/AGAP006228.html"><span class="doc">AGAP006228</span></a> (COEAE2F - carboxylesterase),  <a class="reference internal" href="../../../../../gene/AGAP006229.html"><span class="doc">AGAP006229</span></a> (Vps20 - vacuolar protein sorting 20),  <a class="reference internal" href="../../../../../gene/AGAP006231.html"><span class="doc">AGAP006231</span></a> (serine/threonine-protein phosphatase dullard homolog),  <a class="reference internal" href="../../../../../gene/AGAP006232.html"><span class="doc">AGAP006232</span></a> (peroxin-14),  <a class="reference internal" href="../../../../../gene/AGAP006233.html"><span class="doc">AGAP006233</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006234.html"><span class="doc">AGAP006234</span></a> (protein SHQ1),  <a class="reference internal" href="../../../../../gene/AGAP006235.html"><span class="doc">AGAP006235</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006236.html"><span class="doc">AGAP006236</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006237.html"><span class="doc">AGAP006237</span></a> (Negative elongation factor E),  <a class="reference internal" href="../../../../../gene/AGAP006238.html"><span class="doc">AGAP006238</span></a> (ubiquitin-conjugating enzyme E2 C),  <a class="reference internal" href="../../../../../gene/AGAP006239.html"><span class="doc">AGAP006239</span></a> (Protein TSSC1),  <a class="reference internal" href="../../../../../gene/AGAP006240.html"><span class="doc">AGAP006240</span></a> (splicing factor 3B subunit 2),  <a class="reference internal" href="../../../../../gene/AGAP006241.html"><span class="doc">AGAP006241</span></a> (Innexin inx2),  <a class="reference internal" href="../../../../../gene/AGAP006242.html"><span class="doc">AGAP006242</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006243.html"><span class="doc">AGAP006243</span></a> (phosphatidylinositol-4,5-bisphosphate 4-phosphatase),  <a class="reference internal" href="../../../../../gene/AGAP029069.html"><span class="doc">AGAP029069</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006244.html"><span class="doc">AGAP006244</span></a> (CTL-like protein 1),  <a class="reference internal" href="../../../../../gene/AGAP006245.html"><span class="doc">AGAP006245</span></a> (zinc finger matrin-type protein 2),  <a class="reference internal" href="../../../../../gene/AGAP006246.html"><span class="doc">AGAP006246</span></a> (Sorcin),  <a class="reference internal" href="../../../../../gene/AGAP006247.html"><span class="doc">AGAP006247</span></a> (Survival of motor neuron-related-splicing factor 30),  <a class="reference internal" href="../../../../../gene/AGAP006248.html"><span class="doc">AGAP006248</span></a> (Med10 - Mediator of RNA polymerase II transcription subunit 10),  <a class="reference internal" href="../../../../../gene/AGAP006249.html"><span class="doc">AGAP006249</span></a> (solute carrier family 12 (potassium/chloride transporters), member 8),  <a class="reference internal" href="../../../../../gene/AGAP006250.html"><span class="doc">AGAP006250</span></a> (eukaryotic elongation factor, selenocysteine-tRNA-specific),  <a class="reference internal" href="../../../../../gene/AGAP006251.html"><span class="doc">AGAP006251</span></a> (syntaxin 6),  <a class="reference internal" href="../../../../../gene/AGAP006252.html"><span class="doc">AGAP006252</span></a> (Cysteine-rich venom protein).</p>
<p>The following 11 genes are within 50 kbp of the focal
region: <a class="reference internal" href="../../../../../gene/AGAP028453.html"><span class="doc">AGAP028453</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006214.html"><span class="doc">AGAP006214</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006215.html"><span class="doc">AGAP006215</span></a> (GPRMTH1 - methuselah receptor 1),  <a class="reference internal" href="../../../../../gene/AGAP006216.html"><span class="doc">AGAP006216</span></a> (GPRMTH2 - methuselah receptor 2),  <a class="reference internal" href="../../../../../gene/AGAP006217.html"><span class="doc">AGAP006217</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006218.html"><span class="doc">AGAP006218</span></a> (GPRMTH4 - methuselah receptor 4),  <a class="reference internal" href="../../../../../gene/AGAP006253.html"><span class="doc">AGAP006253</span></a> (Cysteine-rich venom protein),  <a class="reference internal" href="../../../../../gene/AGAP006254.html"><span class="doc">AGAP006254</span></a> (polypeptide N-acetylglucosaminyltransferase),  <a class="reference internal" href="../../../../../gene/AGAP006255.html"><span class="doc">AGAP006255</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP006256.html"><span class="doc">AGAP006256</span></a> (Cad74A),  <a class="reference internal" href="../../../../../gene/AGAP028399.html"><span class="doc">AGAP028399</span></a>.</p>
<div class="figure" id="id1">
<img alt="signal location" src="../../../../../_images/peak_location26.png" />
<p class="caption"><span class="caption-text"><strong>Figure 1</strong>. Location of the signal of selection. Blue markers show the
value of the selection statistic in non-overlapping 20 kbp windows. The
dashed black line shows the fitted peak model. The vertical red bar shows
the inferred focus of the selection signal. The shaded blue area shows the
inferred genomic region affected by the selection event.</span></p>
</div>
<div class="section" id="overlapping-signals">
<h2>Overlapping signals<a class="headerlink" href="#overlapping-signals" title="Permalink to this headline">Â¶</a></h2>
<p>No overlapping signals.</p>
</div>
<div class="section" id="diagnostics">
<h2>Diagnostics<a class="headerlink" href="#diagnostics" title="Permalink to this headline">Â¶</a></h2>
<p>The information below provides some diagnostics from the
<a class="reference internal" href="../../../../../method/peak_modelling.html"><span class="doc">Peak modelling</span></a> algorithm.</p>
<div class="figure" id="id2">
<img alt="../../../../../_images/peak_context26.png" src="../../../../../_images/peak_context26.png" />
<p class="caption"><span class="caption-text"><strong>Figure 2</strong>. Chromosome-wide selection statistic and results from peak
modelling. <strong>a</strong>, TODO. <strong>b</strong>, TODO.</span></p>
</div>
<div class="figure" id="id3">
<img alt="../../../../../_images/peak_targetting26.png" src="../../../../../_images/peak_targetting26.png" />
<p class="caption"><span class="caption-text"><strong>Figure 3</strong>. Diagnostics from targetting the selection signal to a focal
region. TODO.</span></p>
</div>
<div class="figure" id="id4">
<img alt="../../../../../_images/peak_fit26.png" src="../../../../../_images/peak_fit26.png" />
<p class="caption"><span class="caption-text"><strong>Figure 4</strong>. Diagnostics from fitting a peak model to the selection signal.
<strong>a</strong>, TODO. <strong>b</strong>, TODO. <strong>c</strong>, TODO.</span></p>
</div>
<div class="section" id="model-fit-reports">
<h3>Model fit reports<a class="headerlink" href="#model-fit-reports" title="Permalink to this headline">Â¶</a></h3>
<p>Left flank, peak model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">exponential</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 35</span>
    <span class="c1"># data points      = 201</span>
    <span class="c1"># variables        = 3</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.136</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.001</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">1461.033</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">1451.123</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">amplitude</span><span class="p">:</span>   <span class="mf">0.31849123</span> <span class="o">+/-</span> <span class="mf">0.010653</span> <span class="p">(</span><span class="mf">3.35</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">decay</span><span class="p">:</span>       <span class="mf">0.88609190</span> <span class="o">+/-</span> <span class="mf">0.050554</span> <span class="p">(</span><span class="mf">5.71</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">c</span><span class="p">:</span>           <span class="mf">0.04415383</span> <span class="o">+/-</span> <span class="mf">0.002469</span> <span class="p">(</span><span class="mf">5.59</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
    <span class="n">cap</span><span class="p">:</span>         <span class="mi">1</span> <span class="p">(</span><span class="n">fixed</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Correlations</span><span class="p">]]</span> <span class="p">(</span><span class="n">unreported</span> <span class="n">correlations</span> <span class="n">are</span> <span class="o">&lt;</span>  <span class="mf">0.100</span><span class="p">)</span>
    <span class="n">C</span><span class="p">(</span><span class="n">amplitude</span><span class="p">,</span> <span class="n">decay</span><span class="p">)</span>          <span class="o">=</span> <span class="o">-</span><span class="mf">0.574</span>
    <span class="n">C</span><span class="p">(</span><span class="n">decay</span><span class="p">,</span> <span class="n">c</span><span class="p">)</span>                  <span class="o">=</span> <span class="o">-</span><span class="mf">0.531</span>
</pre></div>
</div>
<p>Right flank, peak model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">exponential</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 27</span>
    <span class="c1"># data points      = 193</span>
    <span class="c1"># variables        = 3</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.018</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.000</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">1782.306</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">1772.518</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">amplitude</span><span class="p">:</span>   <span class="mf">0.33485561</span> <span class="o">+/-</span> <span class="mf">0.005315</span> <span class="p">(</span><span class="mf">1.59</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">decay</span><span class="p">:</span>       <span class="mf">0.62195091</span> <span class="o">+/-</span> <span class="mf">0.014950</span> <span class="p">(</span><span class="mf">2.40</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">c</span><span class="p">:</span>           <span class="mf">0.01225445</span> <span class="o">+/-</span> <span class="mf">0.000851</span> <span class="p">(</span><span class="mf">6.94</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
    <span class="n">cap</span><span class="p">:</span>         <span class="mi">1</span> <span class="p">(</span><span class="n">fixed</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Correlations</span><span class="p">]]</span> <span class="p">(</span><span class="n">unreported</span> <span class="n">correlations</span> <span class="n">are</span> <span class="o">&lt;</span>  <span class="mf">0.100</span><span class="p">)</span>
    <span class="n">C</span><span class="p">(</span><span class="n">amplitude</span><span class="p">,</span> <span class="n">decay</span><span class="p">)</span>          <span class="o">=</span> <span class="o">-</span><span class="mf">0.666</span>
    <span class="n">C</span><span class="p">(</span><span class="n">decay</span><span class="p">,</span> <span class="n">c</span><span class="p">)</span>                  <span class="o">=</span> <span class="o">-</span><span class="mf">0.425</span>
</pre></div>
</div>
<p>Left flank, null model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">constant</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 6</span>
    <span class="c1"># data points      = 200</span>
    <span class="c1"># variables        = 1</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.954</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.005</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">1066.983</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">1063.685</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">c</span><span class="p">:</span>   <span class="mf">0.07849146</span> <span class="o">+/-</span> <span class="mf">0.004897</span> <span class="p">(</span><span class="mf">6.24</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
</pre></div>
</div>
<p>Right flank, null model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">constant</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 6</span>
    <span class="c1"># data points      = 192</span>
    <span class="c1"># variables        = 1</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.621</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.003</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">1099.010</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">1095.752</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">c</span><span class="p">:</span>   <span class="mf">0.03677189</span> <span class="o">+/-</span> <span class="mf">0.004114</span> <span class="p">(</span><span class="mf">11.19</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
<div class="section" id="comments">
<h2>Comments<a class="headerlink" href="#comments" title="Permalink to this headline">Â¶</a></h2>
<div id="disqus_thread"></div>
<script>
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript></div>
</div>


    </div>
      
  </div>
</div>
<footer class="footer">
  <div class="container">
    <p class="pull-right">
      <a href="#">Back to top</a>
      
    </p>
    <p>
        &copy; Copyright 2017, Anopheles gambiae 1000 Genomes Consortium.<br/>
      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 1.6.5.<br/>
    </p>
  </div>
</footer>
  </body>
</html>