:orphan:




XPEHH/AOM.GWA/2/1
=================

This page describes a signal of selection found in the
:doc:`../../../../../population/AOM` population
when compared with the :doc:`../../../../../population/GWA` population
using the :doc:`../../../../../method/XPEHH` statistic.The focus of this signal is on chromosome arm
**2L** between positions **3,054,895** and
**3,394,895**.
The evidence supporting this signal is
**strong** (:math:`\Delta_{i}` >= 100 on both flanks).





.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----


The following 26 genes overlap the focal region: :doc:`../../../../../gene/AGAP004751` (condensin-2 complex subunit D3),  :doc:`../../../../../gene/AGAP004752` (tRNA (guanine-N(7)-)-methyltransferase),  :doc:`../../../../../gene/AGAP004753`,  :doc:`../../../../../gene/AGAP004754` (CASPL2 - long caspase (CASPL2)),  :doc:`../../../../../gene/AGAP004755`,  :doc:`../../../../../gene/AGAP004756`,  :doc:`../../../../../gene/AGAP013546` (Vacuolar ATPase assembly integral membrane protein VMA21 homolog),  :doc:`../../../../../gene/AGAP004757`,  :doc:`../../../../../gene/AGAP004758`,  :doc:`../../../../../gene/AGAP004759` (oligoribonuclease),  :doc:`../../../../../gene/AGAP004760`,  :doc:`../../../../../gene/AGAP004761`,  :doc:`../../../../../gene/AGAP004762` (alpha-tocopherol transfer protein-like protein),  :doc:`../../../../../gene/AGAP004763`,  :doc:`../../../../../gene/AGAP004764`,  :doc:`../../../../../gene/AGAP004765` (ATP-dependent RNA helicase DDX1),  :doc:`../../../../../gene/AGAP028129`,  :doc:`../../../../../gene/AGAP004766` (endophilin-A),  :doc:`../../../../../gene/AGAP004767`,  :doc:`../../../../../gene/AGAP004768` (periodic tryptophan protein 1),  :doc:`../../../../../gene/AGAP004769` (Ras-related protein Rap-1b precursor),  :doc:`../../../../../gene/AGAP004770` (female reproductive tract protease GLEANR_896),  :doc:`../../../../../gene/AGAP004771`,  :doc:`../../../../../gene/AGAP004772` (SLIT-ROBO Rho GTPase activating protein),  :doc:`../../../../../gene/AGAP004773` (pyruvate dehydrogenase E1 component subunit alpha, mitochondrial),  :doc:`../../../../../gene/AGAP004774` (host cell factor).



The following 12 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP004742` (Pyruvate carboxylase),  :doc:`../../../../../gene/AGAP004743` (Transmembrane emp24 domain-containing protein),  :doc:`../../../../../gene/AGAP004744` (skap - succinyl-CoA ligase beta subunit),  :doc:`../../../../../gene/AGAP004745` (transcription elongation regulator 1),  :doc:`../../../../../gene/AGAP004746` (Thoc5 - THO complex subunit 5),  :doc:`../../../../../gene/AGAP004747` (Uncharacterized protein C05D11.1),  :doc:`../../../../../gene/AGAP004748`,  :doc:`../../../../../gene/AGAP004749`:sup:`1` (coproporphyrinogen III oxidase),  :doc:`../../../../../gene/AGAP004750` (translation initiation factor 4G),  :doc:`../../../../../gene/AGAP004775` (Xaa-Pro dipeptidase),  :doc:`../../../../../gene/AGAP004776` (pre-mRNA-processing factor SLU7),  :doc:`../../../../../gene/AGAP004777`.


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping selection signals
-----------------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak model :math:`\Delta_{i}`
      - Max. percentile
      - Known locus
    * - :doc:`../../../../../signal/H12/UGS/2/2/index`
      - H12
      - Uganda *An. gambiae*
      - 2L:1,814,895-3,094,895
      - 1,377
      - 100.0%
      - Vgsc
    * - :doc:`../../../../../signal/XPEHH/BFS.GWA/2/1/index`
      - XPEHH
      - Burkina Faso *An. gambiae*
      - 2L:2,234,895-3,274,895
      - 938
      - 99.6%
      - Vgsc
    * - :doc:`../../../../../signal/H12/CMS/2/2/index`
      - H12
      - Cameroon *An. gambiae*
      - 2L:3,054,895-3,614,895
      - 725
      - 99.7%
      - nan
    * - :doc:`../../../../../signal/H12/GAS/2/2/index`
      - H12
      - Gabon *An. gambiae*
      - 2L:2,734,895-3,094,895
      - 325
      - 99.8%
      - nan
    * - :doc:`../../../../../signal/XPEHH/GAS.GWA/2/2/index`
      - XPEHH
      - Gabon *An. gambiae*
      - 2L:3,354,895-3,714,895
      - 274
      - 99.9%
      - nan
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/AOM.GWA/2/1/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/AOM.GWA/2/1/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/AOM.GWA/2/1/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_exponential_peak)
    [[Fit Statistics]]
        # function evals   = 52
        # data points      = 424
        # variables        = 4
        chi-square         = 118.029
        reduced chi-square = 0.281
        Akaike info crit   = -534.212
        Bayesian info crit = -518.013
    [[Variables]]
        center:      0 (fixed)
        amplitude:   3.25255305 +/- 0.102693 (3.16%) (init= 3)
        decay:       2.99999948 +/- 0.375860 (12.53%) (init= 0.5)
        skew:        0.86174784 +/- 0.045384 (5.27%) (init= 0)
        baseline:    2.58358772 +/- 0.068185 (2.64%) (init= 1)
        ceiling:     100 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(decay, baseline)           =  0.798 
        C(skew, baseline)            =  0.333 
        C(amplitude, baseline)       = -0.330 
        C(decay, skew)               =  0.301 
        C(amplitude, skew)           = -0.196 
        C(amplitude, decay)          =  0.167 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 12
        # data points      = 423
        # variables        = 1
        chi-square         = 419.972
        reduced chi-square = 0.995
        Akaike info crit   = -1.039
        Bayesian info crit = 3.008
    [[Variables]]
        c:   3.61942187 +/- 0.048504 (1.34%) (init= 1)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


