:orphan:




H12/CMS/2/3
===========

This page describes a signal of selection found in the
:doc:`../../../../../population/CMS` populationusing the :doc:`../../../../../method/H12` statistic.The focus of this signal is on chromosome arm
**2R** between positions **40,300,000** and
**40,740,000**.
The evidence supporting this signal is
**strong** (:math:`\Delta_{i}` >= 100 on both flanks).





.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----


The following 58 genes overlap the focal region: :doc:`../../../../../gene/AGAP003581`:sup:`1` (D-xylulose reductase A),  :doc:`../../../../../gene/AGAP003582`:sup:`1` (D-xylulose reductase A),  :doc:`../../../../../gene/AGAP003583`:sup:`1` (L-iditol 2-dehydrogenase),  :doc:`../../../../../gene/AGAP003584`:sup:`1` (L-iditol 2-dehydrogenase),  :doc:`../../../../../gene/AGAP003585`,  :doc:`../../../../../gene/AGAP003586` (Phosphate carrier, mitochondrial),  :doc:`../../../../../gene/AGAP013333` (Phosphate carrier, mitochondrial),  :doc:`../../../../../gene/AGAP003587` (solute carrier family 26 (sodium-independent sulfate anion transporter)),  :doc:`../../../../../gene/AGAP003588` (VATH - V-type H -transporting ATPase subunit H),  :doc:`../../../../../gene/AGAP003589` (prolyl-tRNA synthetase),  :doc:`../../../../../gene/AGAP003590`,  :doc:`../../../../../gene/AGAP003591` (LRR-repeat protein 1),  :doc:`../../../../../gene/AGAP003592` (RpLP0 - 60S ribosomal protein LP0),  :doc:`../../../../../gene/AGAP003593` (Protein arginine methyltransferase RmtB),  :doc:`../../../../../gene/AGAP003594`,  :doc:`../../../../../gene/AGAP003595` (rRNA biogenesis protein RRP5),  :doc:`../../../../../gene/AGAP003596`,  :doc:`../../../../../gene/AGAP003597` (histone-lysine N-methyltransferase SUV39H),  :doc:`../../../../../gene/AGAP003598` (template-activating factor I),  :doc:`../../../../../gene/AGAP003599` (F-type H -transporting ATPase oligomycin sensitivity conferral protein),  :doc:`../../../../../gene/AGAP003600` (Elongation of very long chain fatty acids protein),  :doc:`../../../../../gene/AGAP003601` (alpha-1,2-mannosyltransferase),  :doc:`../../../../../gene/AGAP003602` (methyltransferase-like protein 9),  :doc:`../../../../../gene/AGAP003603`,  :doc:`../../../../../gene/AGAP003604`,  :doc:`../../../../../gene/AGAP003605` (cullin 2),  :doc:`../../../../../gene/AGAP003606` (Fatty acyl-CoA reductase),  :doc:`../../../../../gene/AGAP003607` (Cad96Cb, isoform B),  :doc:`../../../../../gene/AGAP003608`:sup:`1` (CYP4AA1 - cytochrome P450),  :doc:`../../../../../gene/AGAP003610`,  :doc:`../../../../../gene/AGAP003611` (Fatty acyl-CoA reductase),  :doc:`../../../../../gene/AGAP003612`,  :doc:`../../../../../gene/AGAP003613` (palmitoyltransferase ZDHHC3),  :doc:`../../../../../gene/AGAP003614`,  :doc:`../../../../../gene/AGAP013196`,  :doc:`../../../../../gene/AGAP013432`,  :doc:`../../../../../gene/AGAP029070`,  :doc:`../../../../../gene/AGAP029099`,  :doc:`../../../../../gene/AGAP029064`,  :doc:`../../../../../gene/AGAP029085`,  :doc:`../../../../../gene/AGAP029083`,  :doc:`../../../../../gene/AGAP003615` (toll-interacting protein),  :doc:`../../../../../gene/AGAP003616`,  :doc:`../../../../../gene/AGAP003618` (serine/threonine-protein kinase LATS1/2),  :doc:`../../../../../gene/AGAP003619` (transcription elongation factor SPT6),  :doc:`../../../../../gene/AGAP003620`,  :doc:`../../../../../gene/AGAP003621` (coiled-coil domain-containing protein 25),  :doc:`../../../../../gene/AGAP003622` (Queuine tRNA-ribosyltransferase catalytic subunit 1),  :doc:`../../../../../gene/AGAP003623` (long-chain acyl-CoA synthetase),  :doc:`../../../../../gene/AGAP003624`,  :doc:`../../../../../gene/AGAP029110`,  :doc:`../../../../../gene/AGAP003626`,  :doc:`../../../../../gene/AGAP013221`,  :doc:`../../../../../gene/AGAP012946`,  :doc:`../../../../../gene/AGAP003627`,  :doc:`../../../../../gene/AGAP003629`,  :doc:`../../../../../gene/AGAP003630`,  :doc:`../../../../../gene/AGAP003631` (GPRGRP2 - putative gastrin/bombesin receptor 2).



The following 3 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP003580`,  :doc:`../../../../../gene/AGAP003632` (U3 small nucleolar RNA-associated protein 14),  :doc:`../../../../../gene/AGAP003633`.


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping selection signals
-----------------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak model :math:`\Delta_{i}`
      - Max. percentile
      - Known locus
    * - :doc:`../../../../../signal/IHS/CMS/2/2/index`
      - IHS
      - Cameroon *An. gambiae*
      - 2R:40,060,000-40,780,000
      - 643
      - 100.0%
      - nan
    * - :doc:`../../../../../signal/IHS/BFS/2/3/index`
      - IHS
      - Burkina Faso *An. gambiae*
      - 2R:40,320,000-40,560,000
      - 557
      - 99.5%
      - nan
    * - :doc:`../../../../../signal/IHS/GNS/2/3/index`
      - IHS
      - Guinea *An. gambiae*
      - 2R:40,200,000-40,420,000
      - 450
      - 99.0%
      - nan
    * - :doc:`../../../../../signal/H12/BFS/2/4/index`
      - H12
      - Burkina Faso *An. gambiae*
      - 2R:40,640,000-41,060,000
      - 384
      - 95.9%
      - nan
    * - :doc:`../../../../../signal/XPEHH/CMS.GWA/2/3/index`
      - XPEHH
      - Cameroon *An. gambiae*
      - 2R:40,680,000-40,760,000
      - 100
      - 96.5%
      - nan
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/CMS/2/3/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/CMS/2/3/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/CMS/2/3/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_exponential_peak)
    [[Fit Statistics]]
        # function evals   = 54
        # data points      = 396
        # variables        = 4
        chi-square         = 0.084
        reduced chi-square = 0.000
        Akaike info crit   = -3341.844
        Bayesian info crit = -3325.919
    [[Variables]]
        center:      0 (fixed)
        amplitude:   0.08320422 +/- 0.002716 (3.27%) (init= 0.5)
        decay:       2.99999974 +/- 0.236898 (7.90%) (init= 0.5)
        skew:       -0.93490345 +/- 0.053920 (5.77%) (init= 0)
        baseline:    0.02065198 +/- 0.001923 (9.31%) (init= 0.03)
        ceiling:     1 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(decay, baseline)           = -0.778 
        C(amplitude, baseline)       = -0.466 
        C(skew, baseline)            = -0.329 
        C(decay, skew)               =  0.307 
        C(amplitude, skew)           =  0.306 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 11
        # data points      = 395
        # variables        = 1
        chi-square         = 0.296
        reduced chi-square = 0.001
        Akaike info crit   = -2840.165
        Bayesian info crit = -2836.186
    [[Variables]]
        c:   0.05095336 +/- 0.001379 (2.71%) (init= 0.03)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


