
Uganda *An. gambiae* | H12 | Chromosome 2 | Signal #2
================================================================================



This page describes a signal of selection found in the
:doc:`/populations/ugs` population using the
:doc:`/methods/h12` statistic.
The inferred focus of this signal is on chromosome arm 2L from
position 2,000,001 to 3,100,000.




The following 62 genes overlap the focal region: :doc:`/genes/AGAP004700`,  :doc:`/genes/AGAP004701`,  :doc:`/genes/AGAP028436`,  :doc:`/genes/AGAP004702`,  :doc:`/genes/AGAP004703` (DNA-directed RNA polymerase III subunit RPC1),  :doc:`/genes/AGAP004704` (COMPASS component SPP1),  :doc:`/genes/AGAP028437`,  :doc:`/genes/AGAP004706`,  :doc:`/genes/AGAP004707` (para - voltage-gated sodium channel),  :doc:`/genes/AGAP004708` (arginyl-tRNA synthetase),  :doc:`/genes/AGAP004709` (mRpL18 - 39S ribosomal protein L18, mitochondrial),  :doc:`/genes/AGAP004710` (ubiquinol-cytochrome c reductase subunit 9),  :doc:`/genes/AGAP004711` (ATP-dependent RNA helicase DDX41),  :doc:`/genes/AGAP004712`,  :doc:`/genes/AGAP004713`,  :doc:`/genes/AGAP004714`,  :doc:`/genes/AGAP004715` (Pyruvate dehydrogenase phosphatase regulatory subunit, mitochondrial),  :doc:`/genes/AGAP004716` (Gr57 - gustatory receptor 57),  :doc:`/genes/AGAP004717`,  :doc:`/genes/AGAP004718`,  :doc:`/genes/AGAP004719` (CLIPC9 - CLIP-domain serine protease),  :doc:`/genes/AGAP004720`,  :doc:`/genes/AGAP004721` (N-acetylglucosamine-6-sulfatase),  :doc:`/genes/AGAP004722`,  :doc:`/genes/AGAP004723` (chromobox protein homolog 1),  :doc:`/genes/AGAP004724` (Intraflagellar transport 74 homolog),  :doc:`/genes/AGAP004725` (eIF3c - Eukaryotic translation initiation factor 3 subunit C),  :doc:`/genes/AGAP004726` (Uncharacterized protein CG3556),  :doc:`/genes/AGAP004727` (Gr25 - gustatory receptor 25),  :doc:`/genes/AGAP004728`,  :doc:`/genes/AGAP004729`,  :doc:`/genes/AGAP004730` (phospholipase A2, venom),  :doc:`/genes/AGAP004731` (secretory phospholipase A2),  :doc:`/genes/AGAP029113`,  :doc:`/genes/AGAP004733`,  :doc:`/genes/AGAP004735`,  :doc:`/genes/AGAP004736` (mitochondrial GTPase 1 homolog),  :doc:`/genes/AGAP004737` (Rhomboid-4, isoform B),  :doc:`/genes/AGAP004738` (IK cytokine),  :doc:`/genes/AGAP004739` (H/ACA ribonucleoprotein complex subunit 4),  :doc:`/genes/AGAP004740`,  :doc:`/genes/AGAP004741` (serine proteinase),  :doc:`/genes/AGAP004742` (Pyruvate carboxylase),  :doc:`/genes/AGAP004743` (Transmembrane emp24 domain-containing protein),  :doc:`/genes/AGAP004744` (skap - succinyl-CoA ligase beta subunit),  :doc:`/genes/AGAP004745` (transcription elongation regulator 1),  :doc:`/genes/AGAP004746` (Thoc5 - THO complex subunit 5),  :doc:`/genes/AGAP004747` (Uncharacterized protein C05D11.1),  :doc:`/genes/AGAP004748`,  :doc:`/genes/AGAP004749` (coproporphyrinogen III oxidase),  :doc:`/genes/AGAP004750` (translation initiation factor 4G),  :doc:`/genes/AGAP004751` (condensin-2 complex subunit D3),  :doc:`/genes/AGAP004752` (tRNA (guanine-N(7)-)-methyltransferase),  :doc:`/genes/AGAP004753`,  :doc:`/genes/AGAP004754` (CASPL2 - long caspase (CASPL2)),  :doc:`/genes/AGAP004755`,  :doc:`/genes/AGAP004756`,  :doc:`/genes/AGAP013546` (Vacuolar ATPase assembly integral membrane protein VMA21 homolog),  :doc:`/genes/AGAP004757`,  :doc:`/genes/AGAP004758`,  :doc:`/genes/AGAP004759` (oligoribonuclease),  :doc:`/genes/AGAP004760`.




The following 4 genes are within 40 kbp of the focal
region: :doc:`/genes/AGAP004696` (exd - Homeobox protein extradenticle),  :doc:`/genes/AGAP004698` (pre-mRNA-splicing factor 38B),  :doc:`/genes/AGAP004699` (RAF proto-oncogene serine/threonine-protein kinase),  :doc:`/genes/AGAP004761`.


.. figure:: signal_location.png
    :alt: signal location

    **Figure 1**. Location of the signal of selection. Blue markers show the
    value of the selection statistic in non-overlapping 20 kbp windows. The
    dashed black line shows the fitted peak model. The vertical red bar shows
    the inferred focus of the selection signal. The shaded blue area shows the
    inferred genomic region affected by the selection event.

Related signals
---------------

Overlapping signals
~~~~~~~~~~~~~~~~~~~

The following selection signals have an inferred focus which overlaps with the
focus of this signal:

.. cssclass:: table-hover
.. csv-table::
    :widths: auto
    :header: Signal, Focus, Score

    :doc:`/signals/h12/bfm/chr2/1/index`,"2L:2520001-2900000",1399
    :doc:`/signals/h12/bfs/chr2/1/index`,"2L:2420001-2720000",1365
    

Adjacent signals
~~~~~~~~~~~~~~~~

The following selection signals have an inferred focus that is immediately
adjacent to the focus of this signal:

.. cssclass:: table-hover
.. csv-table::
    :header: Signal, Chromosome, Start, Stop, Score, Genes

    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234
    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234
    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234
    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234

Nearby signals
~~~~~~~~~~~~~~

The following signals affect a genome region that overlaps with the genome region
affected by this signal:

.. cssclass:: table-hover
.. csv-table::
    :header: Signal, Chromosome, Start, Stop, Score, Genes

    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234
    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234
    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234
    :doc:`/signals/h12/bfs/1/index`, 2L, "2,420,000", "2,460,000", 511.2, AGAP001234

Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`/methods/peak_modelling` procedure.

.. figure:: signal_context.png

    **Figure 2**. Chromosome-wide selection statistic and results from peak
    modelling. **a**, TODO. **b**, TODO.

.. figure:: signal_targetting.png

    **Figure 3**. Diagnostics from targetting the selection signal to a focal
    region. TODO.

.. figure:: signal_fit.png

    **Figure 4**. Diagnostics from fitting a peak model to the selection signal.
    **a**, TODO. **b**, TODO. **c**, TODO.

Model fit reports
~~~~~~~~~~~~~~~~~

Left flank, peak model::

    [[Model]]
        Model(exponential)
    [[Fit Statistics]]
        # function evals   = 27
        # data points      = 312
        # variables        = 3
        chi-square         = 2.825
        reduced chi-square = 0.009
        Akaike info crit   = -1461.813
        Bayesian info crit = -1450.584
    [[Variables]]
        amplitude:   0.79564531 +/- 0.020019 (2.52%) (init= 0.5)
        decay:       1.61494774 +/- 0.111795 (6.92%) (init= 0.5)
        c:           0.10852908 +/- 0.018705 (17.24%) (init= 0.04)
        cap:         1 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(decay, c)                  = -0.863 
        C(amplitude, c)              = -0.578 
        C(amplitude, decay)          =  0.199 


Right flank, peak model::

    [[Model]]
        Model(exponential)
    [[Fit Statistics]]
        # function evals   = 66
        # data points      = 175
        # variables        = 3
        chi-square         = 1.004
        reduced chi-square = 0.006
        Akaike info crit   = -897.210
        Bayesian info crit = -887.716
    [[Variables]]
        amplitude:   0.92953580 +/- 0        (0.00%) (init= 0.5)
        decay:       2.99732687 +/- 0        (0.00%) (init= 0.5)
        c:           5.7147e-11 +/- 0        (0.00%) (init= 0.04)
        cap:         1 (fixed)


Left flank, null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 13
        # data points      = 311
        # variables        = 1
        chi-square         = 17.571
        reduced chi-square = 0.057
        Akaike info crit   = -891.678
        Bayesian info crit = -887.938
    [[Variables]]
        c:   0.42140342 +/- 0.013499 (3.20%) (init= 0.04)


Right flank, null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 12
        # data points      = 174
        # variables        = 1
        chi-square         = 11.963
        reduced chi-square = 0.069
        Akaike info crit   = -463.837
        Bayesian info crit = -460.678
    [[Variables]]
        c:   0.40822393 +/- 0.019934 (4.88%) (init= 0.04)


Comments
--------

.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
