:orphan:




XPEHH/CMS.GWA/2/1
=================

This page describes a signal of selection found in the
:doc:`../../../../../population/CMS` population
when compared with the :doc:`../../../../../population/GWA` population
using the :doc:`../../../../../method/XPEHH` statistic.The focus of this signal is on chromosome arm
**2L** between positions **3,474,895** and
**3,814,895**.
The evidence supporting this signal is
**strong** (:math:`\Delta_{i}` >= 100 on both flanks).





.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----


The following 29 genes overlap the focal region: :doc:`../../../../../gene/AGAP004783` (predicted G-protein coupled receptor GPCR),  :doc:`../../../../../gene/AGAP004784` (fatty acyl-CoA reductase 2),  :doc:`../../../../../gene/AGAP004785` (alpha-tocopherol transfer protein-like),  :doc:`../../../../../gene/AGAP004786` (Pyruvate dehydrogenase E1 component subunit alpha),  :doc:`../../../../../gene/AGAP004787` (Fatty acyl-CoA reductase),  :doc:`../../../../../gene/AGAP004788` (Up-regulated during skeletal muscle growth 5 homolog),  :doc:`../../../../../gene/AGAP004789` (high mobility group 20A),  :doc:`../../../../../gene/AGAP004790` (Up-regulated during skeletal muscle growth 5 homolog),  :doc:`../../../../../gene/AGAP004791` (high mobility group 20A),  :doc:`../../../../../gene/AGAP004792` (StAR-related lipid transfer protein 7),  :doc:`../../../../../gene/AGAP004793` (ornithine--oxo-acid transaminase),  :doc:`../../../../../gene/AGAP004794` (pancreatic triacylglycerol lipase),  :doc:`../../../../../gene/AGAP004795` (RNA polymerase II elongation factor ELL),  :doc:`../../../../../gene/AGAP004797`,  :doc:`../../../../../gene/AGAP004798` (Coatomer protein complex, subunit beta 2),  :doc:`../../../../../gene/AGAP004799`,  :doc:`../../../../../gene/AGAP004800`,  :doc:`../../../../../gene/AGAP004801` (Huntingtin interacting protein),  :doc:`../../../../../gene/AGAP004802`:sup:`1` (4-hydroxyphenylpyruvate dioxygenase),  :doc:`../../../../../gene/AGAP004803`,  :doc:`../../../../../gene/AGAP004804`,  :doc:`../../../../../gene/AGAP004805`,  :doc:`../../../../../gene/AGAP004806` (GALE6 - galectin 6),  :doc:`../../../../../gene/AGAP004807` (GALE7 - galectin 7),  :doc:`../../../../../gene/AGAP004808` (protease m1 zinc metalloprotease),  :doc:`../../../../../gene/AGAP004809` (APN1 - alanyl aminopeptidase N 1),  :doc:`../../../../../gene/AGAP004810` (CTL3 - C-type lectin (CTL)),  :doc:`../../../../../gene/AGAP004811` (CTL1 - C-type lectin (CTL)),  :doc:`../../../../../gene/AGAP004812` (calcium-independent phospholipase A2).



The following 12 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP004775` (Xaa-Pro dipeptidase),  :doc:`../../../../../gene/AGAP004778`,  :doc:`../../../../../gene/AGAP004780` (cyclin-dependent kinase 12/13),  :doc:`../../../../../gene/AGAP004781` (beta-1,4-galactosyltransferase I),  :doc:`../../../../../gene/AGAP004782` (tubulin-specific chaperone D),  :doc:`../../../../../gene/AGAP004813`,  :doc:`../../../../../gene/AGAP004814`,  :doc:`../../../../../gene/AGAP004815`,  :doc:`../../../../../gene/AGAP004816`:sup:`1`,  :doc:`../../../../../gene/AGAP004817` (lig - Protein lingerer),  :doc:`../../../../../gene/AGAP004818` (mRpS16 - 28S ribosomal protein S16, mitochondrial),  :doc:`../../../../../gene/AGAP004819` (receptor expression-enhancing protein 5).


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping selection signals
-----------------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak Model :math:`\Delta_{i}`
      - Max Percentile
      - Known Loci
    * - :doc:`../../../../../signal/H12/CMS/2/2/index`
      - H12
      - Cameroon *An. gambiae*
      - 2L:3,054,895-3,614,895
      - 725
      - 99.7%
      - nan
    * - :doc:`../../../../../signal/XPEHH/AOM.BFM/2/2/index`
      - XPEHH
      - Angola *An. coluzzii*
      - 2L:3,574,895-3,834,895
      - 364
      - 99.9%
      - nan
    * - :doc:`../../../../../signal/XPEHH/GAS.GWA/2/2/index`
      - XPEHH
      - Gabon *An. gambiae*
      - 2L:3,354,895-3,714,895
      - 274
      - 99.9%
      - nan
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GWA/2/1/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GWA/2/1/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GWA/2/1/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_exponential_peak)
    [[Fit Statistics]]
        # function evals   = 59
        # data points      = 549
        # variables        = 4
        chi-square         = 343.584
        reduced chi-square = 0.630
        Akaike info crit   = -249.298
        Bayesian info crit = -232.065
    [[Variables]]
        center:      0 (fixed)
        amplitude:   5.05327004 +/- 0.129550 (2.56%) (init= 3)
        decay:       3          +/- 4.65e-05 (0.00%) (init= 0.5)
        skew:        0.34316648 +/- 0.035400 (10.32%) (init= 0)
        baseline:    1.85888039 +/- 0.126257 (6.79%) (init= 1)
        ceiling:     100 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(decay, baseline)           =  0.898 
        C(amplitude, baseline)       = -0.601 
        C(amplitude, decay)          = -0.299 
        C(skew, baseline)            =  0.203 
        C(amplitude, skew)           = -0.156 
        C(decay, skew)               =  0.156 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 12
        # data points      = 548
        # variables        = 1
        chi-square         = 1388.447
        reduced chi-square = 2.538
        Akaike info crit   = 511.457
        Bayesian info crit = 515.763
    [[Variables]]
        c:   3.63451003 +/- 0.068058 (1.87%) (init= 1)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


