:orphan:




H12/UGS/3/3
===========

This page describes a signal of selection found in the
:doc:`../../../../../population/UGS` populationusing the :doc:`../../../../../method/H12` statistic.The focus of this signal is on chromosome arm
**3R** between positions **44,380,000** and
**45,140,000**.
The evidence supporting this signal is
**strong** (:math:`\Delta_{i}` >= 100 on both flanks).





.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----


The following 40 genes overlap the focal region: :doc:`../../../../../gene/AGAP009859`,  :doc:`../../../../../gene/AGAP009860`,  :doc:`../../../../../gene/AGAP009861` (Profilin protein),  :doc:`../../../../../gene/AGAP009862`,  :doc:`../../../../../gene/AGAP009863` (eukaryotic translation initiation factor 4A, isoform 2),  :doc:`../../../../../gene/AGAP009864`,  :doc:`../../../../../gene/AGAP009865`:sup:`1` (NADH dehydrogenase (ubiquinone) 1 beta subcomplex 9),  :doc:`../../../../../gene/AGAP009866` (dolichol-phosphate mannosyltransferase),  :doc:`../../../../../gene/AGAP009867` (AFG1-family ATPase),  :doc:`../../../../../gene/AGAP009868`:sup:`4` (CPR73 - cuticular protein RR-1 family 73),  :doc:`../../../../../gene/AGAP009869`:sup:`4` (CPR74 - cuticular protein RR-1 family 74),  :doc:`../../../../../gene/AGAP009870`:sup:`4` (CPR151 - cuticular protein RR-2 family 151),  :doc:`../../../../../gene/AGAP009871`:sup:`4` (CPR75 - cuticular protein RR-1 family 75),  :doc:`../../../../../gene/AGAP009872`:sup:`4` (CPR133 - cuticular protein RR-1 family 133),  :doc:`../../../../../gene/AGAP009873`:sup:`4` (CPR153 - cuticular protein RR-2 family 153),  :doc:`../../../../../gene/AGAP028422`,  :doc:`../../../../../gene/AGAP009874`:sup:`4` (CPR76 - cuticular protein RR-1 family 76),  :doc:`../../../../../gene/AGAP009875`:sup:`4` (CPR77 - cuticular protein RR-1 family 77),  :doc:`../../../../../gene/AGAP009876`:sup:`4` (CPR78 - cuticular protein RR-1 family 78),  :doc:`../../../../../gene/AGAP009877`:sup:`4` (CPR79 - cuticular protein RR-1 family 79),  :doc:`../../../../../gene/AGAP009878`:sup:`4` (CPR80 - cuticular protein RR-1 family 80),  :doc:`../../../../../gene/AGAP009879`:sup:`4` (CPR81 - cuticular protein RR-1 family 81),  :doc:`../../../../../gene/AGAP009880` (anaphase-promoting complex subunit 8),  :doc:`../../../../../gene/AGAP009881`,  :doc:`../../../../../gene/AGAP009882` (heat shock factor-binding protein 1),  :doc:`../../../../../gene/AGAP009883`,  :doc:`../../../../../gene/AGAP009884`,  :doc:`../../../../../gene/AGAP009885`,  :doc:`../../../../../gene/AGAP009886` (Med15 - Mediator of RNA polymerase II transcription subunit 15),  :doc:`../../../../../gene/AGAP009887`,  :doc:`../../../../../gene/AGAP009888`,  :doc:`../../../../../gene/AGAP009889` (cabut),  :doc:`../../../../../gene/AGAP009890` (nuclear receptor subfamily 2 group E member 3),  :doc:`../../../../../gene/AGAP009891` (juvenile hormone esterase binding protein),  :doc:`../../../../../gene/AGAP009893` (nocturnin),  :doc:`../../../../../gene/AGAP028648`,  :doc:`../../../../../gene/AGAP009896` (proton-coupled amino acid transporter),  :doc:`../../../../../gene/AGAP009897` (proton-coupled amino acid transporter),  :doc:`../../../../../gene/AGAP009898`,  :doc:`../../../../../gene/AGAP009899`.



The following 17 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP009850` (ABCG7 - ATP-binding cassette transporter (ABC transporter) family G member 7),  :doc:`../../../../../gene/AGAP009851`,  :doc:`../../../../../gene/AGAP009852` (mitochondrial fission 1 protein),  :doc:`../../../../../gene/AGAP009853`:sup:`3` (Gr5 - gustatory receptor 5),  :doc:`../../../../../gene/AGAP009854`:sup:`3` (Gr6 - gustatory receptor 6),  :doc:`../../../../../gene/AGAP009855`:sup:`3` (Gr7 - gustatory receptor 7),  :doc:`../../../../../gene/AGAP009856`:sup:`3` (Gr8 - gustatory receptor 8),  :doc:`../../../../../gene/AGAP009857`:sup:`3` (Gr4 - gustatory receptor 4),  :doc:`../../../../../gene/AGAP009858`:sup:`3` (Gr3 - gustatory receptor 3),  :doc:`../../../../../gene/AGAP009901` (diphosphoinositol-polyphosphate diphosphatase),  :doc:`../../../../../gene/AGAP009902` (E3 ubiquitin-protein ligase RNF146),  :doc:`../../../../../gene/AGAP009903` (tRNA pseudouridine synthase (Fragment)),  :doc:`../../../../../gene/AGAP009904` (tRNA (adenine-N(1)-)-methyltransferase non-catalytic subunit),  :doc:`../../../../../gene/AGAP009905` (PHD finger-like domain-containing protein 5A),  :doc:`../../../../../gene/AGAP009906`,  :doc:`../../../../../gene/AGAP009907` (leukotriene-A4 hydrolase),  :doc:`../../../../../gene/AGAP009908` (mRpL13 - 39S ribosomal protein L13, mitochondrial).


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping selection signals
-----------------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak model :math:`\Delta_{i}`
      - Max. percentile
      - Known locus
    * - :doc:`../../../../../signal/XPEHH/CMS.GAS/3/1/index`
      - XPEHH
      - Cameroon *An. gambiae*
      - 3R:43,920,000-44,560,000
      - 538
      - 99.5%
      - nan
    * - :doc:`../../../../../signal/XPEHH/AOM.BFM/3/1/index`
      - XPEHH
      - Angola *An. coluzzii*
      - 3R:43,860,000-44,500,000
      - 314
      - 99.7%
      - nan
    * - :doc:`../../../../../signal/XPEHH/AOM.GWA/3/1/index`
      - XPEHH
      - Angola *An. coluzzii*
      - 3R:44,320,000-44,480,000
      - 253
      - 99.3%
      - nan
    * - :doc:`../../../../../signal/IHS/AOM/3/1/index`
      - IHS
      - Angola *An. coluzzii*
      - 3R:43,740,000-44,560,000
      - 112
      - 99.2%
      - nan
    * - :doc:`../../../../../signal/XPEHH/UGS.BFS/3/6/index`
      - XPEHH
      - Uganda *An. gambiae*
      - 3R:44,440,000-44,520,000
      - 95
      - 99.7%
      - nan
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/UGS/3/3/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/UGS/3/3/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/UGS/3/3/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_exponential_peak)
    [[Fit Statistics]]
        # function evals   = 38
        # data points      = 486
        # variables        = 4
        chi-square         = 0.142
        reduced chi-square = 0.000
        Akaike info crit   = -3948.051
        Bayesian info crit = -3931.306
    [[Variables]]
        center:      0 (fixed)
        amplitude:   0.06476358 +/- 0.003149 (4.86%) (init= 0.5)
        decay:       2.72448767 +/- 0.334528 (12.28%) (init= 0.5)
        skew:        0.71374081 +/- 0.072551 (10.16%) (init= 0)
        baseline:    0.02600071 +/- 0.002358 (9.07%) (init= 0.03)
        ceiling:     1 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(decay, baseline)           = -0.843 
        C(decay, skew)               = -0.428 
        C(amplitude, baseline)       = -0.420 
        C(skew, baseline)            =  0.299 
        C(amplitude, skew)           = -0.143 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 11
        # data points      = 485
        # variables        = 1
        chi-square         = 0.257
        reduced chi-square = 0.001
        Akaike info crit   = -3655.511
        Bayesian info crit = -3651.327
    [[Variables]]
        c:   0.04636686 +/- 0.001047 (2.26%) (init= 0.03)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


