:orphan:




XPEHH/CMS.GWA/2/2
=================

This page describes a signal of selection found in the
:doc:`../../../../../population/CMS` population
when compared with the :doc:`../../../../../population/GWA` population
using the :doc:`../../../../../method/XPEHH` statistic.The focus of this signal is on chromosome arm
**2R** between positions **28,420,000** and
**28,620,000**.
The evidence supporting this signal is
**strong** (:math:`\Delta_{i}` >= 100 on both flanks).


This signal overlaps the :doc:`../../../../../known-locus/cyp6p`, a genome
region with prior evidence of an association with insecticide resistance and/or recent positive selection in
*Anopheles* mosquitoes.




.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----


The following 25 genes overlap the focal region: :doc:`../../../../../gene/AGAP002859` (solute carrier family 8 (sodium/calcium exchanger)),  :doc:`../../../../../gene/AGAP002862`:sup:`1` (CYP6AA1 - cytochrome P450),  :doc:`../../../../../gene/AGAP013128`:sup:`1` (CYP6AA2 - cytochrome P450),  :doc:`../../../../../gene/AGAP002863`:sup:`1` (COEAE6O - carboxylesterase alpha esterase),  :doc:`../../../../../gene/AGAP002864`:sup:`1` (CYP6P15P - cytochrome P450),  :doc:`../../../../../gene/AGAP002865`:sup:`1` (CYP6P3 - cytochrome P450),  :doc:`../../../../../gene/AGAP002866`:sup:`1` (CYP6P5 - cytochrome P450),  :doc:`../../../../../gene/AGAP002867`:sup:`1` (CYP6P4 - cytochrome P450),  :doc:`../../../../../gene/AGAP002868`:sup:`1` (CYP6P1 - cytochrome P450),  :doc:`../../../../../gene/AGAP002869`:sup:`1` (CYP6P2 - cytochrome P450),  :doc:`../../../../../gene/AGAP002870`:sup:`1` (CYP6AD1 - cytochrome P450),  :doc:`../../../../../gene/AGAP013202`,  :doc:`../../../../../gene/AGAP000586`,  :doc:`../../../../../gene/AGAP002872` (lipase),  :doc:`../../../../../gene/AGAP002873`,  :doc:`../../../../../gene/AGAP013069`,  :doc:`../../../../../gene/AGAP002874`,  :doc:`../../../../../gene/AGAP002875` (protein HEXIM1/2),  :doc:`../../../../../gene/AGAP013244` (adenosine deaminase, tRNA-specific 2, TAD2 homolog),  :doc:`../../../../../gene/AGAP002876` (single-strand selective monofunctional uracil DNA glycosylase),  :doc:`../../../../../gene/AGAP002877` (Tetratricopeptide repeat protein 30 homolog),  :doc:`../../../../../gene/AGAP002878` (Cystatin-like protein),  :doc:`../../../../../gene/AGAP002879` (cathepsin F),  :doc:`../../../../../gene/AGAP002880` (COP9 signalosome complex subunit 5),  :doc:`../../../../../gene/AGAP002881` (GPRNPR1 - putative neuropeptide receptor 1).



The following 3 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP002858` (Sodium/potassium-transporting ATPase subunit alpha),  :doc:`../../../../../gene/AGAP002883`:sup:`1`,  :doc:`../../../../../gene/AGAP013115`.


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping selection signals
-----------------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak Model :math:`\Delta_{i}`
      - Max Percentile
      - Known Loci
    * - :doc:`../../../../../signal/IHS/UGS/2/1/index`
      - IHS
      - Uganda *An. gambiae*
      - 2R:28,280,000-28,700,000
      - 1,661
      - 100.0%
      - Cyp6p
    * - :doc:`../../../../../signal/H12/UGS/2/1/index`
      - H12
      - Uganda *An. gambiae*
      - 2R:28,460,000-28,500,000
      - 1,567
      - 99.2%
      - Cyp6p
    * - :doc:`../../../../../signal/IHS/CMS/2/1/index`
      - IHS
      - Cameroon *An. gambiae*
      - 2R:28,240,000-28,560,000
      - 1,249
      - 100.0%
      - Cyp6p
    * - :doc:`../../../../../signal/IHS/BFS/2/1/index`
      - IHS
      - Burkina Faso *An. gambiae*
      - 2R:28,260,000-28,540,000
      - 1,164
      - 99.1%
      - Cyp6p
    * - :doc:`../../../../../signal/IHS/GNS/2/1/index`
      - IHS
      - Guinea *An. gambiae*
      - 2R:28,440,000-28,700,000
      - 1,134
      - 99.8%
      - Cyp6p
    * - :doc:`../../../../../signal/H12/CMS/2/1/index`
      - H12
      - Cameroon *An. gambiae*
      - 2R:28,460,000-28,560,000
      - 1,124
      - 100.0%
      - Cyp6p
    * - :doc:`../../../../../signal/H12/GNS/2/2/index`
      - H12
      - Guinea *An. gambiae*
      - 2R:28,420,000-28,460,000
      - 1,073
      - 98.5%
      - Cyp6p
    * - :doc:`../../../../../signal/H12/BFS/2/2/index`
      - H12
      - Burkina Faso *An. gambiae*
      - 2R:28,440,000-28,480,000
      - 976
      - 98.4%
      - Cyp6p
    * - :doc:`../../../../../signal/XPEHH/UGS.GWA/2/1/index`
      - XPEHH
      - Uganda *An. gambiae*
      - 2R:28,460,000-28,600,000
      - 758
      - 99.7%
      - Cyp6p
    * - :doc:`../../../../../signal/XPEHH/BFS.GWA/2/3/index`
      - XPEHH
      - Burkina Faso *An. gambiae*
      - 2R:28,420,000-28,500,000
      - 697
      - 99.1%
      - Cyp6p
    * - :doc:`../../../../../signal/XPEHH/BFM.GWA/2/3/index`
      - XPEHH
      - Burkina Faso *An. coluzzii*
      - 2R:28,380,000-28,520,000
      - 495
      - 99.2%
      - Cyp6p
    * - :doc:`../../../../../signal/H12/BFM/2/4/index`
      - H12
      - Burkina Faso *An. coluzzii*
      - 2R:28,420,000-28,520,000
      - 366
      - 98.5%
      - Cyp6p
    * - :doc:`../../../../../signal/H12/AOM/2/6/index`
      - H12
      - Angola *An. coluzzii*
      - 2R:28,440,000-28,480,000
      - 235
      - 97.8%
      - Cyp6p
    * - :doc:`../../../../../signal/XPEHH/CMS.GAS/2/3/index`
      - XPEHH
      - Cameroon *An. gambiae*
      - 2R:28,560,000-28,800,000
      - 191
      - 100.0%
      - nan
    * - :doc:`../../../../../signal/XPEHH/AOM.GWA/2/7/index`
      - XPEHH
      - Angola *An. coluzzii*
      - 2R:28,480,000-28,520,000
      - 92
      - 84.1%
      - Cyp6p
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GWA/2/2/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GWA/2/2/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/XPEHH/CMS.GWA/2/2/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_exponential_peak)
    [[Fit Statistics]]
        # function evals   = 33
        # data points      = 370
        # variables        = 4
        chi-square         = 108.464
        reduced chi-square = 0.296
        Akaike info crit   = -446.021
        Bayesian info crit = -430.367
    [[Variables]]
        center:      0 (fixed)
        amplitude:   4.16492446 +/- 0.136430 (3.28%) (init= 3)
        decay:       0.53986918 +/- 0.032484 (6.02%) (init= 0.5)
        skew:       -0.06988095 +/- 0.054194 (77.55%) (init= 0)
        baseline:    1.44304661 +/- 0.037268 (2.58%) (init= 1)
        ceiling:     100 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(amplitude, decay)          = -0.600 
        C(decay, baseline)           = -0.420 
        C(decay, skew)               = -0.149 
        C(amplitude, baseline)       = -0.137 
        C(amplitude, skew)           =  0.133 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 11
        # data points      = 369
        # variables        = 1
        chi-square         = 524.849
        reduced chi-square = 1.426
        Akaike info crit   = 132.004
        Bayesian info crit = 135.914
    [[Variables]]
        c:   2.11703559 +/- 0.062169 (2.94%) (init= 1)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


