<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <title>Gabon An. gambiae | H12 | Chromosome 2 | Signal #2 &#8212; Ag1000G Selection Atlas</title>
    <link rel="stylesheet" href="../../../../../_static/bootstrap-sphinx.css" type="text/css" />
    <link rel="stylesheet" href="../../../../../_static/pygments.css" type="text/css" />
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:    '../../../../../',
        VERSION:     'DEVELOPMENT VERSION',
        COLLAPSE_INDEX: false,
        FILE_SUFFIX: '.html',
        HAS_SOURCE:  true,
        SOURCELINK_SUFFIX: '.txt'
      };
    </script>
    <script type="text/javascript" src="../../../../../_static/jquery.js"></script>
    <script type="text/javascript" src="../../../../../_static/underscore.js"></script>
    <script type="text/javascript" src="../../../../../_static/doctools.js"></script>
    <script type="text/javascript" src="../../../../../_static/js/jquery-1.11.0.min.js"></script>
    <script type="text/javascript" src="../../../../../_static/js/jquery-fix.js"></script>
    <script type="text/javascript" src="../../../../../_static/bootstrap-3.3.7/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="../../../../../_static/bootstrap-sphinx.js"></script>
    <link rel="index" title="Index" href="../../../../../genindex.html" />
    <link rel="search" title="Search" href="../../../../../search.html" />
<meta charset='utf-8'>
<meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
<meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1'>
<meta name="apple-mobile-web-app-capable" content="yes">

  </head>
  <body>

  <div id="navbar" class="navbar navbar-default navbar-fixed-top">
    <div class="container">
      <div class="navbar-header">
        <!-- .btn-navbar is used as the toggle for collapsed navbar content -->
        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".nav-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a class="navbar-brand" href="../../../../../index.html">
          Ag1000G Selection Atlas</a>
        <span class="navbar-text navbar-version pull-left"><b>DEVELOPMENT VERSION</b></span>
      </div>

        <div class="collapse navbar-collapse nav-collapse">
          <ul class="nav navbar-nav">
            
            
              <li class="dropdown globaltoc-container">
  <a role="button"
     id="dLabelGlobalToc"
     data-toggle="dropdown"
     data-target="#"
     href="../../../../../index.html">Site <b class="caret"></b></a>
  <ul class="dropdown-menu globaltoc"
      role="menu"
      aria-labelledby="dLabelGlobalToc"><ul>
<li class="toctree-l1"><a class="reference internal" href="../../../../../signals.html">All signals</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../../populations.html">Signals by population</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../../../../methods.html">Methods</a></li>
</ul>
</ul>
</li>
              
                <li class="dropdown">
  <a role="button"
     id="dLabelLocalToc"
     data-toggle="dropdown"
     data-target="#"
     href="#">Page <b class="caret"></b></a>
  <ul class="dropdown-menu localtoc"
      role="menu"
      aria-labelledby="dLabelLocalToc"><ul>
<li><a class="reference internal" href="#">Gabon <em>An. gambiae</em> | H12 | Chromosome 2 | Signal #2</a><ul>
<li><a class="reference internal" href="#overlapping-signals">Overlapping signals</a></li>
<li><a class="reference internal" href="#diagnostics">Diagnostics</a><ul>
<li><a class="reference internal" href="#model-fit-reports">Model fit reports</a></li>
</ul>
</li>
<li><a class="reference internal" href="#comments">Comments</a></li>
</ul>
</li>
</ul>
</ul>
</li>
              
            
            
              
                
              
            
            
            
            
          </ul>

          
            
<form class="navbar-form navbar-right" action="../../../../../search.html" method="get">
 <div class="form-group">
  <input type="text" name="q" class="form-control" placeholder="Search" />
 </div>
  <input type="hidden" name="check_keywords" value="yes" />
  <input type="hidden" name="area" value="default" />
</form>
          
        </div>
    </div>
  </div>

<div class="container">
  <div class="row">
    <div class="col-md-12 content">
      
  <div class="section" id="gabon-an-gambiae-h12-chromosome-2-signal-2">
<h1>Gabon <em>An. gambiae</em> | H12 | Chromosome 2 | Signal #2<a class="headerlink" href="#gabon-an-gambiae-h12-chromosome-2-signal-2" title="Permalink to this headline">¶</a></h1>
<p>This page describes a signal of selection found in the
<a class="reference internal" href="../../../../../population/GAS.html"><span class="doc">Gabon An. gambiae</span></a> population using the
<a class="reference internal" href="../../../../../method/H12.html"><span class="doc">H12 (Garud et al. 20XX)</span></a> statistic.The inferred focus of this signal is on chromosome arm
<strong>2L</strong> between positions <strong>2,600,001</strong> and
<strong>2,640,000</strong>.</p>
<p>The following 2 genes overlap the focal region: <a class="reference internal" href="../../../../../gene/AGAP004715.html"><span class="doc">AGAP004715</span></a> (Pyruvate dehydrogenase phosphatase regulatory subunit, mitochondrial),  <a class="reference internal" href="../../../../../gene/AGAP004716.html"><span class="doc">AGAP004716</span></a> (Gr57 - gustatory receptor 57).</p>
<p>The following 4 genes are within 50 kbp of the focal
region: <a class="reference internal" href="../../../../../gene/AGAP004713.html"><span class="doc">AGAP004713</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP004714.html"><span class="doc">AGAP004714</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP004717.html"><span class="doc">AGAP004717</span></a>,  <a class="reference internal" href="../../../../../gene/AGAP004718.html"><span class="doc">AGAP004718</span></a>.</p>
<div class="figure" id="id1">
<img alt="signal location" src="../../../../../_images/peak_location41.png" />
<p class="caption"><span class="caption-text"><strong>Figure 1</strong>. Location of the signal of selection. Blue markers show the
value of the selection statistic in non-overlapping 20 kbp windows. The
dashed black line shows the fitted peak model. The vertical red bar shows
the inferred focus of the selection signal. The shaded blue area shows the
inferred genomic region affected by the selection event.</span></p>
</div>
<div class="section" id="overlapping-signals">
<h2>Overlapping signals<a class="headerlink" href="#overlapping-signals" title="Permalink to this headline">¶</a></h2>
<p>The following selection signals have an inferred focus which overlaps with the
focus of this signal:</p>
<table border="1" class="colwidths-auto table-hover docutils">
<thead valign="bottom">
<tr class="row-odd"><th class="head">Signal</th>
<th class="head">Focus</th>
<th class="head">Score</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td><a class="reference internal" href="../../../BFM/chr2/1/index.html"><span class="doc">Burkina Faso An. coluzzii | H12 | Chromosome 2 | Signal #1</span></a></td>
<td>2L:2,500,001-2,900,000</td>
<td>1736</td>
</tr>
<tr class="row-odd"><td><a class="reference internal" href="../../../BFS/chr2/1/index.html"><span class="doc">Burkina Faso An. gambiae | H12 | Chromosome 2 | Signal #1</span></a></td>
<td>2L:2,420,001-2,720,000</td>
<td>1728</td>
</tr>
<tr class="row-even"><td><a class="reference internal" href="../../../GNS/chr2/1/index.html"><span class="doc">Guinea An. gambiae | H12 | Chromosome 2 | Signal #1</span></a></td>
<td>2L:1,720,001-2,700,000</td>
<td>1554</td>
</tr>
<tr class="row-odd"><td><a class="reference internal" href="../../../UGS/chr2/2/index.html"><span class="doc">Uganda An. gambiae | H12 | Chromosome 2 | Signal #2</span></a></td>
<td>2L:1,940,001-3,080,000</td>
<td>1332</td>
</tr>
<tr class="row-even"><td><a class="reference internal" href="../../../CMS/chr2/2/index.html"><span class="doc">Cameroon An. gambiae | H12 | Chromosome 2 | Signal #2</span></a></td>
<td>2L:2,460,001-2,960,000</td>
<td>635</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="diagnostics">
<h2>Diagnostics<a class="headerlink" href="#diagnostics" title="Permalink to this headline">¶</a></h2>
<p>The information below provides some diagnostics from the
<a class="reference internal" href="../../../../../method/peak_modelling.html"><span class="doc">Peak modelling</span></a> algorithm.</p>
<div class="figure" id="id2">
<img alt="../../../../../_images/peak_context41.png" src="../../../../../_images/peak_context41.png" />
<p class="caption"><span class="caption-text"><strong>Figure 2</strong>. Chromosome-wide selection statistic and results from peak
modelling. <strong>a</strong>, TODO. <strong>b</strong>, TODO.</span></p>
</div>
<div class="figure" id="id3">
<img alt="../../../../../_images/peak_targetting41.png" src="../../../../../_images/peak_targetting41.png" />
<p class="caption"><span class="caption-text"><strong>Figure 3</strong>. Diagnostics from targetting the selection signal to a focal
region. TODO.</span></p>
</div>
<div class="figure" id="id4">
<img alt="../../../../../_images/peak_fit41.png" src="../../../../../_images/peak_fit41.png" />
<p class="caption"><span class="caption-text"><strong>Figure 4</strong>. Diagnostics from fitting a peak model to the selection signal.
<strong>a</strong>, TODO. <strong>b</strong>, TODO. <strong>c</strong>, TODO.</span></p>
</div>
<div class="section" id="model-fit-reports">
<h3>Model fit reports<a class="headerlink" href="#model-fit-reports" title="Permalink to this headline">¶</a></h3>
<p>Left flank, peak model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">exponential</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 40</span>
    <span class="c1"># data points      = 334</span>
    <span class="c1"># variables        = 3</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.270</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.001</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">2371.892</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">2360.459</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">amplitude</span><span class="p">:</span>   <span class="mf">0.20842355</span> <span class="o">+/-</span> <span class="mf">0.013058</span> <span class="p">(</span><span class="mf">6.27</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">decay</span><span class="p">:</span>       <span class="mf">0.17167794</span> <span class="o">+/-</span> <span class="mf">0.016781</span> <span class="p">(</span><span class="mf">9.78</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">c</span><span class="p">:</span>           <span class="mf">0.05999999</span> <span class="o">+/-</span> <span class="mf">0.001723</span> <span class="p">(</span><span class="mf">2.87</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
    <span class="n">cap</span><span class="p">:</span>         <span class="mi">1</span> <span class="p">(</span><span class="n">fixed</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Correlations</span><span class="p">]]</span> <span class="p">(</span><span class="n">unreported</span> <span class="n">correlations</span> <span class="n">are</span> <span class="o">&lt;</span>  <span class="mf">0.100</span><span class="p">)</span>
    <span class="n">C</span><span class="p">(</span><span class="n">amplitude</span><span class="p">,</span> <span class="n">decay</span><span class="p">)</span>          <span class="o">=</span> <span class="o">-</span><span class="mf">0.639</span>
    <span class="n">C</span><span class="p">(</span><span class="n">decay</span><span class="p">,</span> <span class="n">c</span><span class="p">)</span>                  <span class="o">=</span> <span class="o">-</span><span class="mf">0.321</span>
</pre></div>
</div>
<p>Right flank, peak model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">exponential</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 44</span>
    <span class="c1"># data points      = 222</span>
    <span class="c1"># variables        = 3</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.435</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.002</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">1378.217</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">1368.009</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">amplitude</span><span class="p">:</span>   <span class="mf">0.11448034</span> <span class="o">+/-</span> <span class="mf">0.012723</span> <span class="p">(</span><span class="mf">11.11</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">decay</span><span class="p">:</span>       <span class="mf">2.99999887</span> <span class="o">+/-</span> <span class="mf">1.163246</span> <span class="p">(</span><span class="mf">38.77</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.5</span><span class="p">)</span>
    <span class="n">c</span><span class="p">:</span>           <span class="mf">0.05419457</span> <span class="o">+/-</span> <span class="mf">0.013688</span> <span class="p">(</span><span class="mf">25.26</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
    <span class="n">cap</span><span class="p">:</span>         <span class="mi">1</span> <span class="p">(</span><span class="n">fixed</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Correlations</span><span class="p">]]</span> <span class="p">(</span><span class="n">unreported</span> <span class="n">correlations</span> <span class="n">are</span> <span class="o">&lt;</span>  <span class="mf">0.100</span><span class="p">)</span>
    <span class="n">C</span><span class="p">(</span><span class="n">decay</span><span class="p">,</span> <span class="n">c</span><span class="p">)</span>                  <span class="o">=</span>  <span class="mf">0.932</span>
    <span class="n">C</span><span class="p">(</span><span class="n">amplitude</span><span class="p">,</span> <span class="n">c</span><span class="p">)</span>              <span class="o">=</span> <span class="o">-</span><span class="mf">0.717</span>
    <span class="n">C</span><span class="p">(</span><span class="n">amplitude</span><span class="p">,</span> <span class="n">decay</span><span class="p">)</span>          <span class="o">=</span> <span class="o">-</span><span class="mf">0.500</span>
</pre></div>
</div>
<p>Left flank, null model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">constant</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 6</span>
    <span class="c1"># data points      = 333</span>
    <span class="c1"># variables        = 1</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.562</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.002</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">2124.219</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">2120.411</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">c</span><span class="p">:</span>   <span class="mf">0.07080949</span> <span class="o">+/-</span> <span class="mf">0.002253</span> <span class="p">(</span><span class="mf">3.18</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
</pre></div>
</div>
<p>Right flank, null model:</p>
<div class="highlight-default"><div class="highlight"><pre><span></span><span class="p">[[</span><span class="n">Model</span><span class="p">]]</span>
    <span class="n">Model</span><span class="p">(</span><span class="n">constant</span><span class="p">)</span>
<span class="p">[[</span><span class="n">Fit</span> <span class="n">Statistics</span><span class="p">]]</span>
    <span class="c1"># function evals   = 6</span>
    <span class="c1"># data points      = 221</span>
    <span class="c1"># variables        = 1</span>
    <span class="n">chi</span><span class="o">-</span><span class="n">square</span>         <span class="o">=</span> <span class="mf">0.597</span>
    <span class="n">reduced</span> <span class="n">chi</span><span class="o">-</span><span class="n">square</span> <span class="o">=</span> <span class="mf">0.003</span>
    <span class="n">Akaike</span> <span class="n">info</span> <span class="n">crit</span>   <span class="o">=</span> <span class="o">-</span><span class="mf">1304.831</span>
    <span class="n">Bayesian</span> <span class="n">info</span> <span class="n">crit</span> <span class="o">=</span> <span class="o">-</span><span class="mf">1301.433</span>
<span class="p">[[</span><span class="n">Variables</span><span class="p">]]</span>
    <span class="n">c</span><span class="p">:</span>   <span class="mf">0.09433148</span> <span class="o">+/-</span> <span class="mf">0.003505</span> <span class="p">(</span><span class="mf">3.72</span><span class="o">%</span><span class="p">)</span> <span class="p">(</span><span class="n">init</span><span class="o">=</span> <span class="mf">0.03</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>
<div class="section" id="comments">
<h2>Comments<a class="headerlink" href="#comments" title="Permalink to this headline">¶</a></h2>
<div id="disqus_thread"></div>
<script>
(function() { // DON'T EDIT BELOW THIS LINE
var d = document, s = d.createElement('script');
s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
s.setAttribute('data-timestamp', +new Date());
(d.head || d.body).appendChild(s);
})();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript></div>
</div>


    </div>
      
  </div>
</div>
<footer class="footer">
  <div class="container">
    <p class="pull-right">
      <a href="#">Back to top</a>
      
    </p>
    <p>
        &copy; Copyright 2017, Anopheles gambiae 1000 Genomes Consortium.<br/>
      Created using <a href="http://sphinx-doc.org/">Sphinx</a> 1.6.5.<br/>
    </p>
  </div>
</footer>
  </body>
</html>