{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Setup\n",
    "\n",
    "Sets up the environment for Ag1000G Selection Atlas."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<style type=\"text/css\">\n",
       ".container {\n",
       "    width: 100%;\n",
       "}\n",
       "</style>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%%HTML\n",
    "<style type=\"text/css\">\n",
    ".container {\n",
    "    width: 100%;\n",
    "}\n",
    "</style>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# python standard library\n",
    "import sys\n",
    "import os\n",
    "import operator\n",
    "import itertools\n",
    "import collections\n",
    "import functools\n",
    "import glob\n",
    "import csv\n",
    "import datetime\n",
    "import bisect\n",
    "import sqlite3\n",
    "import subprocess\n",
    "import random\n",
    "import gc\n",
    "import shutil\n",
    "import shelve\n",
    "import contextlib\n",
    "import tempfile\n",
    "import math"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "# plotting setup\n",
    "import matplotlib as mpl\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib.colors import ListedColormap\n",
    "from matplotlib.gridspec import GridSpec\n",
    "import seaborn as sns\n",
    "sns.set_context('paper')\n",
    "sns.set_style('darkgrid')\n",
    "# use seaborn defaults\n",
    "rcParams = plt.rcParams\n",
    "rcParams['savefig.jpeg_quality'] = 100"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "%config InlineBackend.figure_formats = {'retina', 'png'}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# general purpose third party packages\n",
    "import numpy as np\n",
    "nnz = np.count_nonzero\n",
    "import scipy\n",
    "import scipy.stats\n",
    "import scipy.spatial.distance\n",
    "import numexpr\n",
    "import h5py\n",
    "import tables\n",
    "import bcolz\n",
    "import dask\n",
    "import dask.array as da\n",
    "import pandas as pd\n",
    "import IPython\n",
    "from IPython.display import clear_output, display, HTML\n",
    "import sklearn\n",
    "import sklearn.decomposition\n",
    "import sklearn.manifold\n",
    "import petl as etl\n",
    "etl.config.display_index_header = True\n",
    "import humanize\n",
    "from humanize import naturalsize, intcomma, intword\n",
    "import zarr\n",
    "from scipy.stats import entropy\n",
    "import lmfit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "import allel"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "sys.path.insert(0, '../agam-report-base/src/python')\n",
    "from util import *\n",
    "import zcache\n",
    "import veff\n",
    "import hapclust\n",
    "%reload_ext autoreload\n",
    "%autoreload 1\n",
    "%aimport rockies\n",
    "\n",
    "ag1k_dir = '../ngs.sanger.ac.uk/production/ag1000g/phase1'\n",
    "from ag1k import phase1_ar3\n",
    "phase1_ar3.init(os.path.join(ag1k_dir, 'AR3'))\n",
    "\n",
    "from ag1k import phase1_ar31\n",
    "phase1_ar31.init(os.path.join(ag1k_dir, 'AR3.1'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "from ag1k import phase1_selection\n",
    "phase1_selection.init(os.path.join(ag1k_dir, 'selection.1.RC2'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table class='petl'>\n",
       "<thead>\n",
       "<tr>\n",
       "<th>0|seqid</th>\n",
       "<th>1|source</th>\n",
       "<th>2|type</th>\n",
       "<th>3|start</th>\n",
       "<th>4|end</th>\n",
       "<th>5|score</th>\n",
       "<th>6|strand</th>\n",
       "<th>7|phase</th>\n",
       "<th>8|ID</th>\n",
       "</tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "<tr>\n",
       "<td>2R</td>\n",
       "<td>VectorBase</td>\n",
       "<td>gene</td>\n",
       "<td style='text-align: right'>6577</td>\n",
       "<td style='text-align: right'>7851</td>\n",
       "<td>.</td>\n",
       "<td>+</td>\n",
       "<td>.</td>\n",
       "<td>AGAP001096</td>\n",
       "</tr>\n",
       "<tr>\n",
       "<td>2R</td>\n",
       "<td>VectorBase</td>\n",
       "<td>gene</td>\n",
       "<td style='text-align: right'>13754</td>\n",
       "<td style='text-align: right'>15149</td>\n",
       "<td>.</td>\n",
       "<td>+</td>\n",
       "<td>.</td>\n",
       "<td>AGAP013094</td>\n",
       "</tr>\n",
       "<tr>\n",
       "<td>2R</td>\n",
       "<td>VectorBase</td>\n",
       "<td>gene</td>\n",
       "<td style='text-align: right'>18554</td>\n",
       "<td style='text-align: right'>19549</td>\n",
       "<td>.</td>\n",
       "<td>+</td>\n",
       "<td>.</td>\n",
       "<td>AGAP001097</td>\n",
       "</tr>\n",
       "<tr>\n",
       "<td>2R</td>\n",
       "<td>VectorBase</td>\n",
       "<td>gene</td>\n",
       "<td style='text-align: right'>20368</td>\n",
       "<td style='text-align: right'>21901</td>\n",
       "<td>.</td>\n",
       "<td>+</td>\n",
       "<td>.</td>\n",
       "<td>AGAP001098</td>\n",
       "</tr>\n",
       "<tr>\n",
       "<td>2R</td>\n",
       "<td>VectorBase</td>\n",
       "<td>gene</td>\n",
       "<td style='text-align: right'>21882</td>\n",
       "<td style='text-align: right'>31394</td>\n",
       "<td>.</td>\n",
       "<td>-</td>\n",
       "<td>.</td>\n",
       "<td>AGAP001099</td>\n",
       "</tr>\n",
       "</tbody>\n",
       "</table>\n",
       "<p><strong>...</strong></p>"
      ],
      "text/plain": [
       "+-------+--------------+--------+-------+-------+-------+--------+-------+--------------+\n",
       "| seqid | source       | type   | start | end   | score | strand | phase | ID           |\n",
       "+=======+==============+========+=======+=======+=======+========+=======+==============+\n",
       "| '2R'  | 'VectorBase' | 'gene' |  6577 |  7851 | '.'   | '+'    | '.'   | 'AGAP001096' |\n",
       "+-------+--------------+--------+-------+-------+-------+--------+-------+--------------+\n",
       "| '2R'  | 'VectorBase' | 'gene' | 13754 | 15149 | '.'   | '+'    | '.'   | 'AGAP013094' |\n",
       "+-------+--------------+--------+-------+-------+-------+--------+-------+--------------+\n",
       "| '2R'  | 'VectorBase' | 'gene' | 18554 | 19549 | '.'   | '+'    | '.'   | 'AGAP001097' |\n",
       "+-------+--------------+--------+-------+-------+-------+--------+-------+--------------+\n",
       "| '2R'  | 'VectorBase' | 'gene' | 20368 | 21901 | '.'   | '+'    | '.'   | 'AGAP001098' |\n",
       "+-------+--------------+--------+-------+-------+-------+--------+-------+--------------+\n",
       "| '2R'  | 'VectorBase' | 'gene' | 21882 | 31394 | '.'   | '-'    | '.'   | 'AGAP001099' |\n",
       "+-------+--------------+--------+-------+-------+-------+--------+-------+--------------+\n",
       "..."
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tbl_genes = etl.cat(*[get_geneset_features(phase1_ar3.geneset_agamp42_fn, chrom).eq('type', 'gene').unpackdict('attributes', ['ID']) \n",
    "                      for chrom in chromosomes])\n",
    "tbl_genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "('2L', 'VectorBase', 'gene', 2358158, 2431617, '.', '+', '.', 'AGAP004707')"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lkp_gene = tbl_genes.recordlookupone('ID')\n",
    "lkp_gene['AGAP004707']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def plot_list_genes_track(namespace, chrom, ax, gene_labels, plot=True, x_loc=None, **kwargs):\n",
    "    if chrom == '2R':\n",
    "        sns.despine(ax=ax, left=True, offset=5)\n",
    "    else:\n",
    "        sns.despine(ax=ax, left=True, offset=5)\n",
    "    if plot:\n",
    "        plot_genes(namespace.genome, namespace.geneset_agamp42_fn, \n",
    "                   chrom, ax=ax, height=.2, label=False, \n",
    "                   barh_kwargs=dict(lw=0.1, alpha=.2))\n",
    "    if chrom == '2R':\n",
    "#         ax.set_ylabel('genes', ha='left', va='center', rotation=0)\n",
    "#         ax.yaxis.set_label_coords(0, 1, transform=ax.transAxes)\n",
    "        ax.set_yticks([])\n",
    "        ax.set_ylabel('')\n",
    "    else:\n",
    "        ax.set_yticks([])\n",
    "        ax.set_ylabel('')\n",
    "    if chrom == 'X':\n",
    "        ax.set_xlabel('Position (Mbp)', ha='left')\n",
    "        ax.xaxis.set_label_coords(1.1, -.87, transform=ax.transAxes)\n",
    "        ax.set_ylabel('Genes', ha='left', va='center', rotation=0)\n",
    "        ax.yaxis.set_label_coords(1.1, 0.5, transform=ax.transAxes)\n",
    "    \n",
    "    for gid in gene_labels:\n",
    "        rec = lkp_gene[gid]\n",
    "        if rec.seqid == chrom:\n",
    "            x = (rec.start + rec.end) / 2\n",
    "            if rec.strand == '+':\n",
    "                y = .9\n",
    "                marker = 'v'\n",
    "                yt = -1\n",
    "            else:\n",
    "                y = .1\n",
    "                marker = '^'\n",
    "                yt = -4\n",
    "            ax.plot([x], [y], marker=marker, mfc='w', mec='k')\n",
    "            ax.annotate(gene_labels[gid], xy=(x, y), xytext=(5, yt), textcoords='offset points',\n",
    "                        fontsize=7, fontstyle='italic')\n",
    "            \n",
    "            \n",
    "    if x_loc is not None:\n",
    "        for _chrom, _pos in x_loc:\n",
    "\n",
    "            if _chrom == chrom:\n",
    "                y = .1\n",
    "                marker = '^'\n",
    "                yt = -4\n",
    "                ax.plot([_pos], [y], marker=marker, mfc='k', mec='k')\n",
    "\n",
    "    xticks = np.arange(0, len(namespace.genome[chrom]), 10000000)\n",
    "    if chrom in {'3R', '2R'}:\n",
    "        xticks = xticks[:-1]\n",
    "    ax.set_xticks(xticks)\n",
    "    ax.set_xticklabels(xticks//1000000)\n",
    "    ax.set_xlim(0, len(namespace.genome[chrom]))\n",
    "    ax.set_ylim(0, 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "def fig_linear_genome(plotf, genome, chromosomes=None, fig=None, \n",
    "                      bottom=0, height=1, width_factor=1.08, chrom_pad=0.035, \n",
    "                      clip_patch_kwargs=None, **kwargs):\n",
    "    if chromosomes is None:\n",
    "        chromosomes = ['2R', '2L', '3R', '3L', 'X']\n",
    "    genome_size = sum(len(genome[chrom]) for chrom in chromosomes)\n",
    "\n",
    "    from matplotlib.path import Path\n",
    "\n",
    "    if fig is None:\n",
    "        fig = plt.figure(figsize=(8, 1))\n",
    "\n",
    "    left = 0\n",
    "\n",
    "    if clip_patch_kwargs is None:\n",
    "        clip_patch_kwargs = dict()\n",
    "    clip_patch_kwargs.setdefault('edgecolor', 'k')\n",
    "    clip_patch_kwargs.setdefault('facecolor', 'none')\n",
    "    clip_patch_kwargs.setdefault('lw', 1)\n",
    "\n",
    "    axs = dict()\n",
    "    for chrom in chromosomes:\n",
    "\n",
    "        # calculate width needed for this chrom\n",
    "        width = len(genome[chrom]) / (genome_size * width_factor)\n",
    "\n",
    "        # create axes\n",
    "        ax = fig.add_axes([left, bottom, width, height])\n",
    "        ax.set_axis_bgcolor((1, 1, 1, 0));\n",
    "        axs[chrom] = ax\n",
    "\n",
    "        # construct clip path\n",
    "        if chrom in {'2R', '3R'}:\n",
    "            verts = [(0.01, 0.02), (0.9, 0.02), (1.01, 0.3), (1.01, 0.7), (0.9, .98), (0.01, .98), (0.01, 0.02)]\n",
    "            codes = [Path.MOVETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.CLOSEPOLY]\n",
    "        elif chrom == \"X\":\n",
    "            verts = [(0.01, 0.02), (0.9, 0.02), (0.99, 0.3), (0.99, 0.7), (0.9, .98), (0.01, .98), (0.01, 0.02)]\n",
    "            codes = [Path.MOVETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.CLOSEPOLY]\n",
    "        else:\n",
    "            verts = [(0.1, 0.02), (.99, 0.02), (.99, .98), (.1, .98), (-0.01, .7), (-0.01, .3), (0.1, 0.02)]\n",
    "            codes = [Path.MOVETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.CLOSEPOLY]\n",
    "        path = Path(verts, codes)\n",
    "        clip_patch = mpl.patches.PathPatch(path, transform=ax.transAxes, **clip_patch_kwargs)\n",
    "\n",
    "        # do the plotting\n",
    "        plotf(chrom=chrom, ax=ax, clip_patch=clip_patch, **kwargs)\n",
    "\n",
    "        # increment left coordinate\n",
    "        left += len(genome[chrom]) / (genome_size * width_factor)\n",
    "        if chrom in {'2L', '3L'}:\n",
    "            left += chrom_pad\n",
    "\n",
    "    return axs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "autosomes = '2R', '2L', '3R', '3L'\n",
    "chromosomes = autosomes + ('X',)\n",
    "\n",
    "\n",
    "class GenomeFigure(object):\n",
    "    \n",
    "    def __init__(self, genome, *args, **kwargs):\n",
    "        self.chromosomes = kwargs.pop('chromosomes', ['2R', '2L', '3R', '3L', 'X'])\n",
    "        maxchrsize = max(np.array(genome[chrom]).size for chrom in self.chromosomes)\n",
    "        fig = plt.figure(*args, **kwargs)\n",
    "        self.fig = fig\n",
    "        self.ax = dict()\n",
    "        for i, chrom in enumerate(self.chromosomes):\n",
    "            ax = fig.add_subplot(3, 2, i+1)\n",
    "            self.ax[chrom] = ax\n",
    "            S = np.array(genome[chrom])\n",
    "            if i % 2 == 1:\n",
    "                sns.despine(ax=ax, offset=10, top=True, left=True, right=False)\n",
    "                ax.set_xlim(0, maxchrsize)\n",
    "                ax.yaxis.tick_right()\n",
    "                ax.yaxis.set_label_position('right')\n",
    "            else:\n",
    "                ax.set_xlim((S.size)-(maxchrsize), S.size)\n",
    "                ax.yaxis.tick_left()\n",
    "                sns.despine(ax=ax, offset=10, top=True, left=False, right=True)\n",
    "            ax.set_xticks(range(0, S.size, int(5e6)))\n",
    "            ax.set_xticklabels(range(0, int(S.size/1e6), 5))\n",
    "            ax.set_title(chrom, fontweight='bold')\n",
    "            ax.xaxis.tick_bottom()\n",
    "        fig.tight_layout()\n",
    "        \n",
    "    def apply(self, f, **kwargs):\n",
    "        chromosomes = kwargs.pop('chromosomes', self.chromosomes)\n",
    "        for chrom in chromosomes:\n",
    "            ax = self.ax[chrom]\n",
    "            f(chrom, ax, **kwargs)\n",
    "        \n",
    "        \n",
    "def subplots(*args, **kwargs):\n",
    "    fig, ax = plt.subplots(*args, **kwargs)\n",
    "    sns.despine(ax=ax, offset=10)\n",
    "    return fig, ax"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
