{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": true,
    "editable": true
   },
   "source": [
    "## Setup\n",
    "\n",
    "Sets up the environment for analyses of the phase 1 selection paper."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# python standard library\n",
    "import sys\n",
    "import os\n",
    "import operator\n",
    "import itertools\n",
    "import collections\n",
    "import functools\n",
    "import glob\n",
    "import csv\n",
    "import datetime\n",
    "import bisect\n",
    "import sqlite3\n",
    "import subprocess\n",
    "import random\n",
    "import gc\n",
    "import shutil\n",
    "import shelve\n",
    "import contextlib\n",
    "import tempfile\n",
    "import math"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# plotting setup\n",
    "import matplotlib as mpl\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib.colors import ListedColormap\n",
    "from matplotlib.gridspec import GridSpec\n",
    "import seaborn as sns\n",
    "sns.set_context('paper')\n",
    "sns.set_style('white')\n",
    "sns.set_style('ticks')\n",
    "rcParams = plt.rcParams\n",
    "# N.B., reduced font size\n",
    "rcParams['font.size'] = 8\n",
    "rcParams['axes.labelsize'] = 10\n",
    "rcParams['xtick.labelsize'] = 10\n",
    "rcParams['ytick.labelsize'] = 10\n",
    "rcParams['legend.fontsize'] = 10\n",
    "rcParams['axes.linewidth'] = .5\n",
    "rcParams['lines.linewidth'] = .5\n",
    "rcParams['patch.linewidth'] = .5\n",
    "rcParams['ytick.direction'] = 'out'\n",
    "rcParams['xtick.direction'] = 'out'\n",
    "rcParams['savefig.jpeg_quality'] = 100\n",
    "rcParams['lines.markeredgewidth'] = .5"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# general purpose third party packages\n",
    "import numpy as np\n",
    "nnz = np.count_nonzero\n",
    "import scipy\n",
    "import scipy.stats\n",
    "import scipy.spatial.distance\n",
    "import numexpr\n",
    "import h5py\n",
    "import tables\n",
    "import bcolz\n",
    "import dask\n",
    "import dask.array as da\n",
    "import pandas as pd\n",
    "import IPython\n",
    "from IPython.display import clear_output, display, HTML\n",
    "import sklearn\n",
    "import sklearn.decomposition\n",
    "import sklearn.manifold\n",
    "import petl as etl\n",
    "etl.config.display_index_header = True\n",
    "import humanize\n",
    "from humanize import naturalsize, intcomma, intword\n",
    "import zarr\n",
    "from scipy.stats import entropy\n",
    "import lmfit"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import allel"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "sys.path.insert(0, '../agam-report-base/src/python')\n",
    "from util import *\n",
    "import zcache\n",
    "import veff\n",
    "import hapclust\n",
    "\n",
    "ag1k_dir = '../../data/release'\n",
    "from ag1k import phase1_ar3\n",
    "phase1_ar3.init(os.path.join(ag1k_dir, 'phase1.AR3'))\n",
    "\n",
    "from ag1k import phase1_ar31\n",
    "phase1_ar31.init(os.path.join(ag1k_dir, 'phase1.AR3.1'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "from ag1k import phase1_selection\n",
    "phase1_selection.init(os.path.join(ag1k_dir, 'phase1.selection.1.RC2'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "tbl_genes = etl.cat(*[get_geneset_features(phase1_ar3.geneset_agamp42_fn, chrom).eq('type', 'gene').unpackdict('attributes', ['ID']) \n",
    "                      for chrom in chromosomes])\n",
    "tbl_genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "outputs": [],
   "source": [
    "lkp_gene = tbl_genes.recordlookupone('ID')\n",
    "lkp_gene['AGAP004707']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def plot_list_genes_track(namespace, chrom, ax, gene_labels, plot=True, x_loc=None, **kwargs):\n",
    "    if chrom == '2R':\n",
    "        sns.despine(ax=ax, left=True, offset=5)\n",
    "    else:\n",
    "        sns.despine(ax=ax, left=True, offset=5)\n",
    "    if plot:\n",
    "        plot_genes(namespace.genome, namespace.geneset_agamp42_fn, \n",
    "                   chrom, ax=ax, height=.2, label=False, \n",
    "                   barh_kwargs=dict(lw=0.1, alpha=.2))\n",
    "    if chrom == '2R':\n",
    "#         ax.set_ylabel('genes', ha='left', va='center', rotation=0)\n",
    "#         ax.yaxis.set_label_coords(0, 1, transform=ax.transAxes)\n",
    "        ax.set_yticks([])\n",
    "        ax.set_ylabel('')\n",
    "    else:\n",
    "        ax.set_yticks([])\n",
    "        ax.set_ylabel('')\n",
    "    if chrom == 'X':\n",
    "        ax.set_xlabel('Position (Mbp)', ha='left')\n",
    "        ax.xaxis.set_label_coords(1.1, -.87, transform=ax.transAxes)\n",
    "        ax.set_ylabel('Genes', ha='left', va='center', rotation=0)\n",
    "        ax.yaxis.set_label_coords(1.1, 0.5, transform=ax.transAxes)\n",
    "    \n",
    "    for gid in gene_labels:\n",
    "        rec = lkp_gene[gid]\n",
    "        if rec.seqid == chrom:\n",
    "            x = (rec.start + rec.end) / 2\n",
    "            if rec.strand == '+':\n",
    "                y = .9\n",
    "                marker = 'v'\n",
    "                yt = -1\n",
    "            else:\n",
    "                y = .1\n",
    "                marker = '^'\n",
    "                yt = -4\n",
    "            ax.plot([x], [y], marker=marker, mfc='w', mec='k')\n",
    "            ax.annotate(gene_labels[gid], xy=(x, y), xytext=(5, yt), textcoords='offset points',\n",
    "                        fontsize=7, fontstyle='italic')\n",
    "            \n",
    "            \n",
    "    if x_loc is not None:\n",
    "        for _chrom, _pos in x_loc:\n",
    "\n",
    "            if _chrom == chrom:\n",
    "                y = .1\n",
    "                marker = '^'\n",
    "                yt = -4\n",
    "                ax.plot([_pos], [y], marker=marker, mfc='k', mec='k')\n",
    "\n",
    "    xticks = np.arange(0, len(namespace.genome[chrom]), 10000000)\n",
    "    if chrom in {'3R', '2R'}:\n",
    "        xticks = xticks[:-1]\n",
    "    ax.set_xticks(xticks)\n",
    "    ax.set_xticklabels(xticks//1000000)\n",
    "    ax.set_xlim(0, len(namespace.genome[chrom]))\n",
    "    ax.set_ylim(0, 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def fig_linear_genome(plotf, genome, chromosomes=None, fig=None, \n",
    "                      bottom=0, height=1, width_factor=1.08, chrom_pad=0.035, \n",
    "                      clip_patch_kwargs=None, **kwargs):\n",
    "    if chromosomes is None:\n",
    "        chromosomes = ['2R', '2L', '3R', '3L', 'X']\n",
    "    genome_size = sum(len(genome[chrom]) for chrom in chromosomes)\n",
    "\n",
    "    from matplotlib.path import Path\n",
    "\n",
    "    if fig is None:\n",
    "        fig = plt.figure(figsize=(8, 1))\n",
    "\n",
    "    left = 0\n",
    "\n",
    "    if clip_patch_kwargs is None:\n",
    "        clip_patch_kwargs = dict()\n",
    "    clip_patch_kwargs.setdefault('edgecolor', 'k')\n",
    "    clip_patch_kwargs.setdefault('facecolor', 'none')\n",
    "    clip_patch_kwargs.setdefault('lw', 1)\n",
    "\n",
    "    axs = dict()\n",
    "    for chrom in chromosomes:\n",
    "\n",
    "        # calculate width needed for this chrom\n",
    "        width = len(genome[chrom]) / (genome_size * width_factor)\n",
    "\n",
    "        # create axes\n",
    "        ax = fig.add_axes([left, bottom, width, height])\n",
    "        ax.set_axis_bgcolor((1, 1, 1, 0));\n",
    "        axs[chrom] = ax\n",
    "\n",
    "        # construct clip path\n",
    "        if chrom in {'2R', '3R'}:\n",
    "            verts = [(0.01, 0.02), (0.9, 0.02), (1.01, 0.3), (1.01, 0.7), (0.9, .98), (0.01, .98), (0.01, 0.02)]\n",
    "            codes = [Path.MOVETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.CLOSEPOLY]\n",
    "        elif chrom == \"X\":\n",
    "            verts = [(0.01, 0.02), (0.9, 0.02), (0.99, 0.3), (0.99, 0.7), (0.9, .98), (0.01, .98), (0.01, 0.02)]\n",
    "            codes = [Path.MOVETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.CLOSEPOLY]\n",
    "        else:\n",
    "            verts = [(0.1, 0.02), (.99, 0.02), (.99, .98), (.1, .98), (-0.01, .7), (-0.01, .3), (0.1, 0.02)]\n",
    "            codes = [Path.MOVETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.LINETO, Path.CLOSEPOLY]\n",
    "        path = Path(verts, codes)\n",
    "        clip_patch = mpl.patches.PathPatch(path, transform=ax.transAxes, **clip_patch_kwargs)\n",
    "\n",
    "        # do the plotting\n",
    "        plotf(chrom=chrom, ax=ax, clip_patch=clip_patch, **kwargs)\n",
    "\n",
    "        # increment left coordinate\n",
    "        left += len(genome[chrom]) / (genome_size * width_factor)\n",
    "        if chrom in {'2L', '3L'}:\n",
    "            left += chrom_pad\n",
    "\n",
    "    return axs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "autosomes = '2R', '2L', '3R', '3L'\n",
    "chromosomes = autosomes + ('X',)\n",
    "\n",
    "\n",
    "class GenomeFigure(object):\n",
    "    \n",
    "    def __init__(self, genome, *args, **kwargs):\n",
    "        self.chromosomes = kwargs.pop('chromosomes', ['2R', '2L', '3R', '3L', 'X'])\n",
    "        maxchrsize = max(np.array(genome[chrom]).size for chrom in self.chromosomes)\n",
    "        fig = plt.figure(*args, **kwargs)\n",
    "        self.fig = fig\n",
    "        self.ax = dict()\n",
    "        for i, chrom in enumerate(self.chromosomes):\n",
    "            ax = fig.add_subplot(3, 2, i+1)\n",
    "            self.ax[chrom] = ax\n",
    "            S = np.array(genome[chrom])\n",
    "            if i % 2 == 1:\n",
    "                sns.despine(ax=ax, offset=10, top=True, left=True, right=False)\n",
    "                ax.set_xlim(0, maxchrsize)\n",
    "                ax.yaxis.tick_right()\n",
    "                ax.yaxis.set_label_position('right')\n",
    "            else:\n",
    "                ax.set_xlim((S.size)-(maxchrsize), S.size)\n",
    "                ax.yaxis.tick_left()\n",
    "                sns.despine(ax=ax, offset=10, top=True, left=False, right=True)\n",
    "            ax.set_xticks(range(0, S.size, int(5e6)))\n",
    "            ax.set_xticklabels(range(0, int(S.size/1e6), 5))\n",
    "            ax.set_title(chrom, fontweight='bold')\n",
    "            ax.xaxis.tick_bottom()\n",
    "        fig.tight_layout()\n",
    "        \n",
    "    def apply(self, f, **kwargs):\n",
    "        chromosomes = kwargs.pop('chromosomes', self.chromosomes)\n",
    "        for chrom in chromosomes:\n",
    "            ax = self.ax[chrom]\n",
    "            f(chrom, ax, **kwargs)\n",
    "        \n",
    "        \n",
    "def subplots(*args, **kwargs):\n",
    "    fig, ax = plt.subplots(*args, **kwargs)\n",
    "    sns.despine(ax=ax, offset=10)\n",
    "    return fig, ax"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def filter_zarr(data, start, stop, field, pos_field=\"POS\"):\n",
    "    \n",
    "    pos = allel.SortedIndex(data[pos_field])\n",
    "    ix = pos.locate_range(start, stop)\n",
    "    \n",
    "    return pos[ix], data[field][ix]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def compute_gain(target_attr, pos, haps, plots=False):\n",
    "\n",
    "    n_classes = np.bincount(target_attr).size\n",
    "    n_haps = haps.shape[1]\n",
    "\n",
    "    # compute entropy for the target attribute\n",
    "    target_freqs = np.bincount(target_attr, minlength=n_classes) / target_attr.shape[0]\n",
    "    target_entropy = entropy(target_freqs)\n",
    "\n",
    "    # setup output array\n",
    "    gain = np.zeros(pos.shape[0])\n",
    "\n",
    "    # work through the variants one by one\n",
    "    for i in range(pos.shape[0]):\n",
    "\n",
    "        # pull out the attribute data\n",
    "        attr = haps[i]\n",
    "\n",
    "        # split on attribute value and compute entropies for each split\n",
    "        split_entropy = 0\n",
    "        for v in 0, 1, 2:\n",
    "            split = target_attr[attr == v]\n",
    "            if split.shape[0] == 0:\n",
    "                continue\n",
    "            split_freqs = np.bincount(split, minlength=n_classes) / split.shape[0]\n",
    "            split_entropy += (split.shape[0] / n_haps) * scipy.stats.entropy(split_freqs)\n",
    "\n",
    "        # compute and store gain\n",
    "        gain[i] = target_entropy - split_entropy\n",
    "        \n",
    "    if plots:\n",
    "        make_gain_plots(gain, pos, haps)\n",
    "        \n",
    "        \n",
    "    return gain"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def make_gain_plots(gain, pos, haps):\n",
    "    \n",
    "    \n",
    "    fig, axes = plt.subplots(nrows=2, sharex=False, sharey=True, figsize=(9, 6))\n",
    "    \n",
    "    for ax in axes:\n",
    "        sns.despine(ax=ax, offset=5)\n",
    "    \n",
    "    ax1, ax2 = axes\n",
    "    ax1.scatter(pos/1e6, gain, marker=\"o\", alpha=0.1, color=\"k\")\n",
    "    ax1.set_ylabel(\"Information gain\")\n",
    "    ax1.set_xlabel(\"position (Mbp)\")\n",
    "    \n",
    "    ac = haps.count_alleles()\n",
    "    freq = ac.to_frequencies()[:, 0]\n",
    "    ax2.scatter(freq, gain, marker=\"o\", alpha=0.1, color=\"k\")\n",
    "    ax2.set_ylabel(\"Information gain\")\n",
    "    ax2.set_xlabel(\"Alt freq.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# assume access to callset pass.\n",
    "def retrieve_allele_freqs(chrom, pos):\n",
    "    \n",
    "    af = os.path.join(ag1k_dir, \"phase1.AR3/extras/allele_frequencies.h5\")\n",
    "    affh = h5py.File(af, \"r\")\n",
    "    \n",
    "    callset_pass = phase1_ar3.callset_pass\n",
    "    \n",
    "    reference = callset_pass[chrom][\"variants/REF\"][:].astype(\"<U1\")\n",
    "    alternate = callset_pass[chrom][\"variants/ALT\"][:].astype(\"<U1\")\n",
    "    positions = allel.SortedIndex(callset_pass[chrom][\"variants/POS\"][:])\n",
    "    \n",
    "    ix = positions.locate_keys(pos)\n",
    "    \n",
    "    afq_df = pd.DataFrame(index=pos, columns=[\"REF\", \"ALT\"] + sorted(affh[chrom].keys()))\n",
    "    afq_df[\"REF\"] = np.compress(ix, reference, 0)    \n",
    "    afq_df[\"ALT\"] = np.compress(ix, alternate, 0)\n",
    "\n",
    "    for grp in affh[chrom].keys():\n",
    "        afq_df[grp] = np.compress(ix, affh[chrom][grp], 0)\n",
    "    \n",
    "    return afq_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# function that computes frequencies in each class\n",
    "def compute_class_frequencies(gain, pos, haps, target_attr, names, plot=True):\n",
    "    \n",
    "    assert pos.size == haps.shape[0] == gain.size\n",
    "    assert np.bincount(target_attr).size == len(names) + 1\n",
    "\n",
    "    subpops = {\"susceptible\": np.where(Y == 0)[0]}\n",
    "    \n",
    "    for ix, na in enumerate(names):\n",
    "        subpops[na] = np.where(target_attr == (ix + 1))[0]\n",
    "\n",
    "    ac = haps.count_alleles_subpops(subpops=subpops, max_allele=1)\n",
    "    af_ = pd.DataFrame(index=pos, columns=[\"susceptible\"] + names)\n",
    "    \n",
    "    for k in subpops.keys():\n",
    "        af_[k] = ac[k].to_frequencies()[:, 1]\n",
    "        \n",
    "    af_[\"variance\"] = af_.var(1)\n",
    "    af_[\"gain\"] = gain\n",
    "    \n",
    "    if plot:\n",
    "        fig, ax = plt.subplots()\n",
    "        ax.scatter(af_.variance, af_.gain)\n",
    "        ax.set_xlabel(\"Variance\")\n",
    "        ax.set_ylabel(\"Information gain\")\n",
    "                \n",
    "    return af_"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.Data import IUPACData\n",
    "# create reverse dictionary\n",
    "iupac = IUPACData.ambiguous_dna_values.copy()\n",
    "iupac.pop(\"X\")\n",
    "iupac_rev_iub = {\"\".join(sorted(v)): k for k, v in iupac.items()}\n",
    "\n",
    "# function that generates string for assay design.\n",
    "def generate_assay_design_string(chrom, pos_list, buf=1):\n",
    "    \n",
    "    # position start, stop. This is *inclusive*\n",
    "    start, stop = pos_list[0] - buf, pos_list[-1] + buf\n",
    "    \n",
    "    # get reference loci: This is 0 based, so we subtract 1. Then +1 for inclusivity.\n",
    "    reference_seq = np.array(list(phase1_ar3.genome[chrom][(start - 1):stop].upper()))\n",
    "    \n",
    "    # find other variants in region\n",
    "    all_variant_pos = allel.SortedIndex(phase1_ar3.callset[chrom][\"variants/POS\"])\n",
    "\n",
    "    # this is inclusive and 0 based \n",
    "    ix = all_variant_pos.locate_range(start, stop)\n",
    "    \n",
    "    reg_gt = allel.GenotypeArray(phase1_ar3.callset[chrom][\"calldata/genotype\"][ix])\n",
    "    reg_pos = all_variant_pos[ix]\n",
    "    reg_ref = phase1_ar3.callset[chrom][\"variants/REF\"][ix]\n",
    "    reg_alt = phase1_ar3.callset[chrom][\"variants/ALT\"][ix]\n",
    "    \n",
    "    span_pos = allel.SortedIndex(range(start, stop + 1))\n",
    "        \n",
    "    # loop through and apply upac codes.\n",
    "    for p, gt, rref, ralt in zip(reg_pos, reg_gt, reg_ref, reg_alt):\n",
    "        \n",
    "        # determine if singleton\n",
    "        gv = allel.GenotypeVector(gt)\n",
    "        x = gv.to_allele_counts(max_allele=3).sum(0)\n",
    "        not_singleton = np.sort(x)[:-1].sum() > 1\n",
    "        \n",
    "        # check fasta agrees with variant data.\n",
    "        qq = phase1_ar3.genome[chrom][int(p - 1)]        \n",
    "        assert rref.decode() == qq.upper(), (rref, \"vs\", qq)\n",
    "        \n",
    "        if not_singleton:\n",
    "            # work out upac code... and replace\n",
    "            qix = span_pos.locate_key(p)\n",
    "            c = rref + b''.join(ralt) \n",
    "            iupacstring = \"\".join(sorted(list(c.decode())))\n",
    "            reference_seq[qix] = iupac_rev_iub[iupacstring]\n",
    "            \n",
    "     # determine accessibilty\n",
    "    is_accessible = phase1_ar3.accessibility[chrom][\"is_accessible\"][start - 1:stop]    \n",
    "    for v in np.where(~is_accessible)[0]:\n",
    "        reference_seq[v] = reference_seq[v].lower()\n",
    "    \n",
    "    qlist = []\n",
    "    for p in pos_list:\n",
    "        \n",
    "        pix = reg_pos.locate_key(p)\n",
    "        \n",
    "        fref = reg_ref[pix].astype(\"<U1\")\n",
    "        falt = reg_alt[pix].astype(\"<U1\")\n",
    "        \n",
    "        # assert biallelic\n",
    "        assert falt[1] == falt[2] == \"\"\n",
    "        \n",
    "        centre = \"[{0}|{1}]\".format(fref, falt[0])\n",
    "        \n",
    "        ix_span = span_pos.locate_key(p)                            \n",
    "                                    \n",
    "        upstream = \"\".join(reference_seq[(ix_span - buf):ix_span])\n",
    "        dnstream = \"\".join(reference_seq[(ix_span + 1):(ix_span + 1 + buf)])\n",
    "        \n",
    "        qlist.append(upstream + centre + dnstream)\n",
    "    \n",
    "    return pd.Series(qlist, index=pos_list, name=\"assay_string\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def locate_functional_snps(chrom, haplotypes, positions, callset_pass, clusters, ref_ix):\n",
    "    \n",
    "    # larger as includes biallelics\n",
    "    pos_pass = allel.SortedIndex(phase1_ar3.callset_pass[chrom]['variants/POS'][:])\n",
    "    pass_loc = pos_pass.locate_range(positions[0], positions[-1])\n",
    "    \n",
    "    ann_pass = phase1_ar3.callset_pass[chrom]['variants/ANN'][:][['Annotation', 'HGVS_p', 'Gene_ID']][pass_loc]\n",
    "\n",
    "    # need to remove missing overlaps ie multiallelics\n",
    "    ann_region = ann_pass.compress(np.in1d(pos_pass[pass_loc], positions), 0)\n",
    "    \n",
    "    # all haplotype positions should be in the pos pass array.\n",
    "    assert np.all(np.in1d(positions, positions)), \"missing positions\"\n",
    "    \n",
    "    loc_region_missense = (ann_region['Annotation'] == b'missense_variant')        \n",
    "\n",
    "    # define types of variants to include in EHH analysis - should be mostly neutral\n",
    "    loc_type_neutral = ((ann_region['Annotation'] == b'intergenic_region') | \n",
    "                        (ann_region['Annotation'] == b'intron_variant') |\n",
    "                        (ann_region['Annotation'] == b'downstream_gene_variant') |\n",
    "                        (ann_region['Annotation'] == b'upstream_gene_variant') |\n",
    "                        (ann_region['Annotation'] == b'synonymous_variant') |\n",
    "                        (ann_region['Annotation'] == b'3_prime_UTR_variant') |\n",
    "                        (ann_region['Annotation'] == b'5_prime_UTR_variant') \n",
    "                        )\n",
    "    \n",
    "    expected_size = positions.size\n",
    "    for d in [haplotypes, ann_region]:\n",
    "        assert d.shape[0] == expected_size, \"wrong shape\"\n",
    "        \n",
    "    print(\"{0:.3f} of muations are neutral\".format(loc_type_neutral.mean()))\n",
    "    \n",
    "    ref_haps = haplotypes.take(ref_ix, axis=1)\n",
    "    unswept_freq = ref_haps.count_alleles(1).to_frequencies()[:, 1]\n",
    "    \n",
    "    swept_f = {}\n",
    "    for k, v in clusters.items():\n",
    "        swept_haps = haplotypes.take(v, axis=1)\n",
    "        swept_f[k] = swept_haps.count_alleles(1).to_frequencies()[:, 1]\n",
    "    swept_freq = np.column_stack(swept_f.values())\n",
    "        \n",
    "    # return swept_freq, unswept_f\n",
    "    \n",
    "    # interesting if *all* clusters above 0.9 / below 0.1\n",
    "    # AND reference is more than x away\n",
    "    maf_ok = np.any(swept_freq > 0.9, 1) | np.any(swept_freq < 0.1, 1) \n",
    "    \n",
    "    diff = np.abs(swept_freq - np.expand_dims(unswept_freq, axis=1))\n",
    "    \n",
    "    \n",
    "    diff_ok = np.any(diff > 0.25, 1)\n",
    "    \n",
    "    pif = maf_ok & diff_ok\n",
    "    \n",
    "    \n",
    "    print(\"diff\", diff[pif][0])\n",
    "    print(\"swept\", swept_freq[pif][0])\n",
    "    print(\"unswept\", unswept_freq[pif][0])\n",
    "    \n",
    "           \n",
    "    # position, whether functional\n",
    "    pos_pif = positions[pif]\n",
    "    neutral = loc_type_neutral[pif]\n",
    "\n",
    "    func_changes = pd.DataFrame(index=pos_pif)\n",
    "    func_changes[\"ref_af\"] = unswept_freq[pif]\n",
    "    for k, v in swept_f.items():\n",
    "        print(k, pos_pif.size, v.size)\n",
    "        func_changes[k + \"_af\"] = v[pif]\n",
    "    func_changes[\"annotation\"] = ann_region[pif][\"Annotation\"].astype(\"str\")\n",
    "    func_changes[\"change\"] = ann_region[pif][\"HGVS_p\"].astype(\"str\")\n",
    "    func_changes[\"is_neutral\"] = neutral\n",
    "\n",
    "    func_changes[\"gene\"] = ann_region[pif][\"Gene_ID\"].astype(\"str\")\n",
    "\n",
    "    return func_changes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from scipy.cluster.hierarchy import _convert_to_double\n",
    "from scipy.spatial import distance\n",
    "from scipy.cluster.hierarchy import _hierarchy\n",
    "\n",
    "def plot_dendrogram(zhier, ax, method='complete', color_threshold=0, above_threshold_color='k'):\n",
    "    \n",
    "    # plot dendrogram\n",
    "    sns.despine(ax=ax, offset=5, bottom=True, top=False)\n",
    "    r = scipy.cluster.hierarchy.dendrogram(zhier, no_labels=True, count_sort=True, \n",
    "                                           color_threshold=color_threshold, \n",
    "                                           above_threshold_color=above_threshold_color,\n",
    "                                           ax=ax)\n",
    "    xmin, xmax = ax.xaxis.get_data_interval()\n",
    "    xticklabels = np.array(list(range(0, h.shape[1], 200)) + [h.shape[1]])\n",
    "    xticks = xticklabels / h.shape[1]\n",
    "    xticks = (xticks * (xmax - xmin)) + xmin\n",
    "    ax.set_xticks(xticks)\n",
    "    ax.set_xticklabels(xticklabels)\n",
    "    ax.set_xlabel('Haplotypes')\n",
    "    ax.xaxis.set_label_position('top')\n",
    "    ax.set_ylim(bottom=-.0001)\n",
    "    \n",
    "    ax.set_ylabel(r'$d_{xy}$')\n",
    "    ax.autoscale(axis='x', tight=True)\n",
    "    return z, r"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# draw an annotated h12 plot containing 2 populations\n",
    "def draw_dbl_h12_plot(chrom, start, stop, pop1, pop2, gene_colors, ylim=0.5, path=None):\n",
    "\n",
    "    gs = GridSpec(2, 1, height_ratios=[1.0, 1.0])\n",
    "    fig = plt.figure(figsize=(10, 2))\n",
    "\n",
    "    ax1 = plt.subplot(gs[0])\n",
    "    ax2 = plt.subplot(gs[1], sharex=ax1)\n",
    "    sns.despine(ax=ax1)\n",
    "    sns.despine(ax=ax2)\n",
    "\n",
    "    max_pos = {}\n",
    "    for i, pop in enumerate([pop1, pop2]):\n",
    "\n",
    "        h12_p, h12_v = filter_zarr(\n",
    "            phase1_selection.hstats_raw[chrom][pop], start, stop, \"h12\", \"pos\")\n",
    "        if i == 1:\n",
    "            h12_v = 0 - h12_v\n",
    "        ax1.fill_between(h12_p[::10], h12_v[::10], \n",
    "                         label=pop, color=phase1_ar3.pop_colors[pop], alpha=1)\n",
    "        ax1.set_ylim((-0.51, 0.51))\n",
    "        \n",
    "        max_pos[pop] = h12_p[np.nanargmax(h12_v)]\n",
    "\n",
    "    ax1.xaxis.set_visible(False)\n",
    "    \n",
    "    q = np.arange(-ylim, ylim+0.01, 0.5)\n",
    "    ax1.set_yticks(q)\n",
    "    ax1.set_yticklabels(np.abs(q))\n",
    "    ax1.set_ylabel(r\"$h_{12}$\", rotation=0)\n",
    "    ax1.yaxis.set_label_coords(0.03, 0.8)\n",
    "    ax1.tick_params(length=2)\n",
    "    ax1.legend(ncol=2, loc=(0.75, 0.81))\n",
    "\n",
    "    plot_genes(phase1_ar3.genome, phase1_ar3.geneset_agamp42_fn,\n",
    "               chrom, start, stop, label=False, labels=None, label_unnamed=False, ax=ax2)\n",
    "\n",
    "    for gid in gene_colors:\n",
    "            rec = lkp_gene[gid]\n",
    "            color = gene_colors[gid]\n",
    "            if rec.seqid == chrom:\n",
    "                x = (rec.start + rec.end) / 2\n",
    "                if rec.strand == '+':\n",
    "                    y = .9\n",
    "                    marker = 'v'\n",
    "                    yt = -1\n",
    "                else:\n",
    "                    y = .1\n",
    "                    marker = '^'\n",
    "                    yt = -4\n",
    "                ax2.plot([x], [y], marker=marker, mfc=color, mec='k')\n",
    "\n",
    "    ax2.set_xticklabels([\"{0:.1f}\".format(x*1e-6) for x in  ax2.get_xticks()])\n",
    "    ax2.tick_params(length=2)\n",
    "    ax2.yaxis.set_label_coords(0.06, 0.92)\n",
    "    ax2.set_xlabel(\"Genomic Position (Mbp)\")\n",
    "    \n",
    "    if path is not None:\n",
    "        fig.savefig(path, dpi=500, bbox_inches=\"tight\")\n",
    "        \n",
    "    return max_pos"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def truspan(cluster, r):\n",
    "    # get the index of the cluster haps in the dendrogram list of all haps\n",
    "    cluster_leaves = sorted([r['leaves'].index(i) for i in cluster])\n",
    "    # are these indices monotonic - they should be!\n",
    "    x = np.asarray(cluster_leaves)\n",
    "    dx = np.diff(x)\n",
    "    mon = np.all(dx == 1)\n",
    "    assert mon\n",
    "    return min(cluster_leaves), max(cluster_leaves)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def draw_hap_cluster_plot(z, r, cluster_labels, vspans, df_haplotypes=phase1_ar31.df_haplotypes,\n",
    "                          path=None):\n",
    "\n",
    "    gs = GridSpec(3, 1, height_ratios=[5.0, 1.0, 0.6])\n",
    "    fig = plt.figure(figsize=(10, 2))\n",
    "\n",
    "    ax1 = plt.subplot(gs[0])\n",
    "    sns.despine(ax=ax1, offset=5, bottom=True, top=False)\n",
    "    _ = plot_dendrogram(z, ax1)\n",
    "\n",
    "    ax_pops = fig.add_subplot(gs[1])\n",
    "\n",
    "    #if hap_pops is None:\n",
    "    hap_pops = df_haplotypes.population.values\n",
    "\n",
    "    x = hap_pops.take(r['leaves'])\n",
    "    hap_clrs = [phase1_ar3.pop_colors[p] for p in x]\n",
    "    ax_pops.broken_barh(xranges=[(i, 1) for i in range(h.shape[1])], \n",
    "                        yrange=(0, 1),\n",
    "                        color=hap_clrs);\n",
    "    sns.despine(ax=ax_pops, offset=5, left=True, bottom=True)\n",
    "    \n",
    "    ax_pops.set_xticks([])\n",
    "    ax_pops.set_yticks([])\n",
    "    ax_pops.set_xlim(0, h.shape[1])\n",
    "    ax_pops.yaxis.set_label_position('left')\n",
    "    ax_pops.set_ylabel('Population', rotation=0, ha='right', va='center')\n",
    "\n",
    "    # cluster brackets\n",
    "    ax_clu = fig.add_subplot(gs[2])\n",
    "    sns.despine(ax=ax_clu, bottom=True, left=True)\n",
    "    ax_clu.set_xlim(0, h.shape[1])\n",
    "    ax_clu.set_ylim(0, 1)\n",
    "    for lbl, (xmin, xmax) in zip(cluster_labels, vspans):\n",
    "        if lbl:\n",
    "            # hack to get the \"fraction\" right, which controls length of bracket arms\n",
    "            fraction = -20 / (xmax - xmin)\n",
    "            ax_clu.annotate(\"\", ha='left', va='center',\n",
    "                            xy=(xmin, 1), xycoords='data',\n",
    "                            xytext=(xmax, 1), textcoords='data',\n",
    "                            arrowprops=dict(arrowstyle=\"-\",\n",
    "                                            connectionstyle=\"bar,fraction=%.4f\" % fraction,\n",
    "                                            ),\n",
    "                            )\n",
    "            ax_clu.text((xmax + xmin)/2, 0.1, lbl, va='top', ha='center')\n",
    "            ax_pops.vlines([xmin, xmax], 0, 1, linestyle=':')\n",
    "\n",
    "    ax_clu.set_xticks([])\n",
    "    ax_clu.set_yticks([])\n",
    "\n",
    "    if path is not None:\n",
    "        fig.savefig(path, dpi=500, bbox_inches=\"tight\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "prior_loci = \"../../data/analysis/20161101-selection-candidate-loci\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "func_annot = \"type1_metabolic\", \"type2_target_site\", \"type3_behavioural\", \"type4_cuticular\"\n",
    "#func_annot_lab = [\"m'bolic\", \"target s.\", \"behav'l\", \"cutic'r\"]\n",
    "func_annot_lab = [\"I\", \"II\", \"III\", \"IV\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "# load annotations\n",
    "\n",
    "gene_classes = {}\n",
    "gene_table = {}\n",
    "\n",
    "for an in func_annot:\n",
    "    gene_classes[an] = pd.read_csv(\n",
    "        os.path.join(prior_loci, an + \".txt\"), \n",
    "        sep=\"\\t\")[\"Gene stable ID\"].tolist()\n",
    "    \n",
    "    df = pd.DataFrame(index=gene_classes[an], columns=[\"classification\"])\n",
    "    df.classification = an\n",
    "    df.index.name = \"gene_id\"\n",
    "    gene_table[an] = df\n",
    "    \n",
    "gene_table = pd.concat(gene_table).reset_index(level=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "gene_table"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "source": [
    "## File paths"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "fstem_scan_plot = \"../artwork/case{case}/scan_{analysis_id}.png\"\n",
    "fstem_scan_plot2 = \"../artwork/case{case}/scan2_{analysis_id}.png\"\n",
    "fstem_tree_plot = \"../artwork/case{case}/pwtree_{analysis_id}.png\"\n",
    "fstem_cluster_plot = \"../artwork/case{case}/cluster_{analysis_id}.png\"\n",
    "fstem_switch_plot = \"../artwork/case{case}/switch_{analysis_id}.png\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "fstem_dis_snps = \"../tables/case{case}/dis_snps_{analysis_id}.txt\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
