:orphan:




IHS/BFM/X/1
===========

This page describes a signal of selection found in the
:doc:`../../../../../population/BFM` populationusing the :doc:`../../../../../method/IHS` statistic.The focus of this signal is on chromosome arm
**X** between positions **14,760,000** and
**15,620,000**.
The evidence supporting this signal is
**strong** (:math:`\Delta_{i}` >= 100 on both flanks).





.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----


The following 41 genes overlap the focal region: :doc:`../../../../../gene/AGAP000806` (Angiopoietin-like 1),  :doc:`../../../../../gene/AGAP000807` (helix-loop-helix transcription factor),  :doc:`../../../../../gene/AGAP000808` (DNA damage-regulated autophagy modulator protein 2),  :doc:`../../../../../gene/AGAP013022`,  :doc:`../../../../../gene/AGAP000809` (Proteasome 26S non-ATPase subunit 10),  :doc:`../../../../../gene/AGAP000810`,  :doc:`../../../../../gene/AGAP000812` (calcium binding protein),  :doc:`../../../../../gene/AGAP000813`:sup:`1` (Frataxin homolog, mitochondrial),  :doc:`../../../../../gene/AGAP000814`,  :doc:`../../../../../gene/AGAP000815` (INTB - integrin beta subunit),  :doc:`../../../../../gene/AGAP000817`,  :doc:`../../../../../gene/AGAP000816`,  :doc:`../../../../../gene/AGAP013474`,  :doc:`../../../../../gene/AGAP013173`,  :doc:`../../../../../gene/AGAP013424`,  :doc:`../../../../../gene/AGAP012997`,  :doc:`../../../../../gene/AGAP000818`:sup:`1` (CYP9K1 - cytochrome P450),  :doc:`../../../../../gene/AGAP000819` (nuclear receptor subfamily 2 group E member (Tailless)),  :doc:`../../../../../gene/AGAP000820`:sup:`4` (CPR125 - cuticular protein RR-2 family 125),  :doc:`../../../../../gene/AGAP000821`,  :doc:`../../../../../gene/AGAP000822`,  :doc:`../../../../../gene/AGAP000823` (CD81 antigen),  :doc:`../../../../../gene/AGAP000824` (bone morphogenetic protein 5),  :doc:`../../../../../gene/AGAP000825`,  :doc:`../../../../../gene/AGAP000826` (cap-specific mRNA (nucleoside-2'-O-)-methyltransferase 1),  :doc:`../../../../../gene/AGAP000829` (calpain-15),  :doc:`../../../../../gene/AGAP000830` (CASPS7 - short caspase 7),  :doc:`../../../../../gene/AGAP000831` (DnaJ homolog subfamily C member 25),  :doc:`../../../../../gene/AGAP000832` (Derlin-2/3),  :doc:`../../../../../gene/AGAP000833` (MIP - myoinhibitory-like peptide),  :doc:`../../../../../gene/AGAP000834`,  :doc:`../../../../../gene/AGAP000835`,  :doc:`../../../../../gene/AGAP028655`,  :doc:`../../../../../gene/AGAP013506`,  :doc:`../../../../../gene/AGAP000840` (amiloride-sensitive sodium channel, other),  :doc:`../../../../../gene/AGAP000841` (Ras-related protein Rab-40),  :doc:`../../../../../gene/AGAP013101`,  :doc:`../../../../../gene/AGAP000842` (NADH dehydrogenase (ubiquinone) 1 alpha subcomplex, assembly factor 1),  :doc:`../../../../../gene/AGAP000843` (cardiolipin synthase),  :doc:`../../../../../gene/AGAP000844` (Progestin and adipoQ receptor family member 4),  :doc:`../../../../../gene/AGAP000847` (CDC-like kinase).



The following 4 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP000801`:sup:`2` (GLURIIb - ionotropic receptor GLURIIb),  :doc:`../../../../../gene/AGAP000803`:sup:`2` (GLURIIa - ionotropic receptor GLURIIa),  :doc:`../../../../../gene/AGAP000804`:sup:`1` (GPXH2 - glutathione peroxidase 2),  :doc:`../../../../../gene/AGAP000805` (BTB/POZ domain-containing protein KCTD16).


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping selection signals
-----------------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak Model :math:`\Delta_{i}`
      - Max Percentile
      - Known Loci
    * - :doc:`../../../../../signal/H12/BFM/X/1/index`
      - H12
      - Burkina Faso *An. coluzzii*
      - X:15,100,000-15,300,000
      - 1,267
      - 100.0%
      - nan
    * - :doc:`../../../../../signal/XPEHH/BFS.UGS/X/1/index`
      - XPEHH
      - Burkina Faso *An. gambiae*
      - X:14,960,000-15,140,000
      - 961
      - 99.9%
      - nan
    * - :doc:`../../../../../signal/XPEHH/BFS.GWA/X/1/index`
      - XPEHH
      - Burkina Faso *An. gambiae*
      - X:15,060,000-15,220,000
      - 866
      - 99.6%
      - nan
    * - :doc:`../../../../../signal/XPEHH/BFM.GWA/X/1/index`
      - XPEHH
      - Burkina Faso *An. coluzzii*
      - X:15,060,000-15,260,000
      - 852
      - 99.4%
      - nan
    * - :doc:`../../../../../signal/H12/BFS/X/1/index`
      - H12
      - Burkina Faso *An. gambiae*
      - X:15,100,000-15,240,000
      - 815
      - 98.5%
      - nan
    * - :doc:`../../../../../signal/XPEHH/BFM.BFS/X/1/index`
      - XPEHH
      - Burkina Faso *An. coluzzii*
      - X:15,580,000-15,800,000
      - 718
      - 99.9%
      - nan
    * - :doc:`../../../../../signal/IHS/GNS/X/1/index`
      - IHS
      - Guinea *An. gambiae*
      - X:15,240,000-15,780,000
      - 553
      - 99.3%
      - nan
    * - :doc:`../../../../../signal/IHS/UGS/X/1/index`
      - IHS
      - Uganda *An. gambiae*
      - X:14,640,000-15,360,000
      - 517
      - 96.7%
      - nan
    * - :doc:`../../../../../signal/XPEHH/BFM.AOM/X/1/index`
      - XPEHH
      - Burkina Faso *An. coluzzii*
      - X:15,040,000-15,180,000
      - 511
      - 100.0%
      - nan
    * - :doc:`../../../../../signal/IHS/BFS/X/2/index`
      - IHS
      - Burkina Faso *An. gambiae*
      - X:14,660,000-14,980,000
      - 477
      - 99.9%
      - nan
    * - :doc:`../../../../../signal/H12/GNS/X/1/index`
      - H12
      - Guinea *An. gambiae*
      - X:14,960,000-15,160,000
      - 419
      - 97.8%
      - nan
    * - :doc:`../../../../../signal/H12/UGS/X/1/index`
      - H12
      - Uganda *An. gambiae*
      - X:15,160,000-15,480,000
      - 348
      - 94.9%
      - nan
    * - :doc:`../../../../../signal/XPEHH/GAS.GWA/X/1/index`
      - XPEHH
      - Gabon *An. gambiae*
      - X:14,460,000-14,860,000
      - 301
      - 98.5%
      - nan
    * - :doc:`../../../../../signal/XPEHH/UGS.CMS/X/1/index`
      - XPEHH
      - Uganda *An. gambiae*
      - X:15,240,000-15,460,000
      - 284
      - 99.9%
      - nan
    * - :doc:`../../../../../signal/XPEHH/AOM.GWA/X/2/index`
      - XPEHH
      - Angola *An. coluzzii*
      - X:14,500,000-15,180,000
      - 228
      - 98.5%
      - nan
    * - :doc:`../../../../../signal/XPEHH/UGS.BFS/X/1/index`
      - XPEHH
      - Uganda *An. gambiae*
      - X:14,960,000-15,320,000
      - 204
      - 89.8%
      - nan
    * - :doc:`../../../../../signal/IHS/CMS/X/1/index`
      - IHS
      - Cameroon *An. gambiae*
      - X:15,520,000-15,560,000
      - 162
      - 96.6%
      - nan
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/IHS/BFM/X/1/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/IHS/BFM/X/1/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/IHS/BFM/X/1/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_gaussian)
    [[Fit Statistics]]
        # function evals   = 43
        # data points      = 314
        # variables        = 4
        chi-square         = 153.412
        reduced chi-square = 0.495
        Akaike info crit   = -216.908
        Bayesian info crit = -201.910
    [[Variables]]
        center:      0 (fixed)
        amplitude:   4.81018313 +/- 0.149632 (3.11%) (init= 3)
        sigma:       2.04839171 +/- 0.071299 (3.48%) (init= 0.5)
        skew:       -0.22832452 +/- 0.026888 (11.78%) (init= 0)
        baseline:    2.58879421 +/- 0.067752 (2.62%) (init= 1)
        ceiling:     100 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(sigma, baseline)           = -0.615 
        C(amplitude, sigma)          = -0.450 
        C(amplitude, baseline)       = -0.193 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 12
        # data points      = 313
        # variables        = 1
        chi-square         = 770.857
        reduced chi-square = 2.471
        Akaike info crit   = 284.107
        Bayesian info crit = 287.853
    [[Variables]]
        c:   3.85278548 +/- 0.088845 (2.31%) (init= 1)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


