:orphan:




H12/GAS/2/2
===========

This page describes a signal of selection found in the
:doc:`../../../../../population/GAS` populationusing the :doc:`../../../../../method/H12` statistic.The focus of this signal is on chromosome arm
**2L** between positions **2,734,895** and
**3,094,895**.
The evidence supporting this signal is
**strong** (:math:`\Delta_{i}` >= 100 on both flanks).

.. raw:: html
    :file: peak_location.html

.. raw:: html

    <div class='bokeh-figure figure'><p class='caption'>
    <strong>Signal location</strong>. Blue markers show the values of the selection statistic.
    The dashed black line shows the fitted peak model. The shaded red area shows the focus of the
    selection signal. The shaded blue area shows the genomic region in linkage with the
    selection event. Use the mouse wheel or the controls at the top right of the plot to zoom
    in, and hover over genes to see gene names and descriptions.
    </p></div>

Genes
-----




The following 41 genes overlap the focal region: :doc:`../../../../../gene/AGAP004720`,  :doc:`../../../../../gene/AGAP004721` (N-acetylglucosamine-6-sulfatase),  :doc:`../../../../../gene/AGAP004722`,  :doc:`../../../../../gene/AGAP004723` (chromobox protein homolog 1),  :doc:`../../../../../gene/AGAP004724` (Intraflagellar transport 74 homolog),  :doc:`../../../../../gene/AGAP004725` (eIF3c - Eukaryotic translation initiation factor 3 subunit C),  :doc:`../../../../../gene/AGAP004726` (Uncharacterized protein CG3556),  :doc:`../../../../../gene/AGAP004727`:sup:`3` (Gr25 - gustatory receptor 25),  :doc:`../../../../../gene/AGAP004728`,  :doc:`../../../../../gene/AGAP004729`,  :doc:`../../../../../gene/AGAP004730` (phospholipase A2, venom),  :doc:`../../../../../gene/AGAP004731` (secretory phospholipase A2),  :doc:`../../../../../gene/AGAP029113`,  :doc:`../../../../../gene/AGAP004733`,  :doc:`../../../../../gene/AGAP004735`,  :doc:`../../../../../gene/AGAP004736` (mitochondrial GTPase 1 homolog),  :doc:`../../../../../gene/AGAP004737` (Rhomboid-4, isoform B),  :doc:`../../../../../gene/AGAP004738` (IK cytokine),  :doc:`../../../../../gene/AGAP004739` (H/ACA ribonucleoprotein complex subunit 4),  :doc:`../../../../../gene/AGAP004740`,  :doc:`../../../../../gene/AGAP004741` (serine proteinase),  :doc:`../../../../../gene/AGAP004742` (Pyruvate carboxylase),  :doc:`../../../../../gene/AGAP004743` (Transmembrane emp24 domain-containing protein),  :doc:`../../../../../gene/AGAP004744` (skap - succinyl-CoA ligase beta subunit),  :doc:`../../../../../gene/AGAP004745` (transcription elongation regulator 1),  :doc:`../../../../../gene/AGAP004746` (Thoc5 - THO complex subunit 5),  :doc:`../../../../../gene/AGAP004747` (Uncharacterized protein C05D11.1),  :doc:`../../../../../gene/AGAP004748`,  :doc:`../../../../../gene/AGAP004749`:sup:`1` (coproporphyrinogen III oxidase),  :doc:`../../../../../gene/AGAP004750` (translation initiation factor 4G),  :doc:`../../../../../gene/AGAP004751` (condensin-2 complex subunit D3),  :doc:`../../../../../gene/AGAP004752` (tRNA (guanine-N(7)-)-methyltransferase),  :doc:`../../../../../gene/AGAP004753`,  :doc:`../../../../../gene/AGAP004754` (CASPL2 - long caspase (CASPL2)),  :doc:`../../../../../gene/AGAP004755`,  :doc:`../../../../../gene/AGAP004756`,  :doc:`../../../../../gene/AGAP013546` (Vacuolar ATPase assembly integral membrane protein VMA21 homolog),  :doc:`../../../../../gene/AGAP004757`,  :doc:`../../../../../gene/AGAP004758`,  :doc:`../../../../../gene/AGAP004759` (oligoribonuclease),  :doc:`../../../../../gene/AGAP004760`.




The following 3 genes are within 50 kbp of the focal
region: :doc:`../../../../../gene/AGAP004718`,  :doc:`../../../../../gene/AGAP004719` (CLIPC9 - CLIP-domain serine protease),  :doc:`../../../../../gene/AGAP004761`.


Key to insecticide resistance candidate gene types: :sup:`1` metabolic;
:sup:`2` target-site; :sup:`3` behavioural; :sup:`4` cuticular.

Overlapping signals
-------------------

The following selection signals have a focus which overlaps with the
focus of this signal.

.. cssclass:: table-hover
.. list-table::
    :widths: auto
    :header-rows: 1

    * - Signal
      - Statistic
      - Population
      - Focus
      - Peak Model :math:`\Delta_{i}`
      - Max Percentile
    * - :doc:`../../../../../signal/H12/BFM/2/1/index`
      - H12
      - Burkina Faso *An. coluzzii*
      - 2L:2,474,895-2,894,895
      - 1,800
      - 99.9%
    * - :doc:`../../../../../signal/H12/UGS/2/2/index`
      - H12
      - Uganda *An. gambiae*
      - 2L:1,814,895-3,094,895
      - 1,377
      - 100.0%
    * - :doc:`../../../../../signal/XPEHH/BFS.GWA/2/1/index`
      - XPEHH
      - Burkina Faso *An. gambiae*
      - 2L:2,234,895-3,274,895
      - 938
      - 99.6%
    * - :doc:`../../../../../signal/H12/CMS/2/2/index`
      - H12
      - Cameroon *An. gambiae*
      - 2L:3,054,895-3,614,895
      - 725
      - 99.7%
    




Diagnostics
-----------

The information below provides some diagnostics from the
:doc:`../../../../../method/peak_modelling` algorithm.

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/GAS/2/2/peak_finding.png"/>
    <p class="caption"><strong>Selection signal in context</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/GAS/2/2/peak_targetting.png"/>
    <p class="caption"><strong>Peak targetting</strong>. @@TODO</p>
    </div>

.. raw:: html

    <div class="figure">
    <img src="../../../../../_static/data/signal/H12/GAS/2/2/peak_fit.png"/>
    <p class="caption"><strong>Peak fitting diagnostics</strong>. @@TODO</p>
    </div>

Model fit reports
~~~~~~~~~~~~~~~~~

Peak model::

    [[Model]]
        Model(skewed_gaussian)
    [[Fit Statistics]]
        # function evals   = 66
        # data points      = 556
        # variables        = 4
        chi-square         = 0.715
        reduced chi-square = 0.001
        Akaike info crit   = -3692.909
        Bayesian info crit = -3675.626
    [[Variables]]
        center:      0 (fixed)
        amplitude:   0.12185633 +/- 0.006364 (5.22%) (init= 0.5)
        sigma:       0.69162740 +/- 0.041820 (6.05%) (init= 0.5)
        skew:       -0.99999910 +/- 0.064121 (6.41%) (init= 0)
        baseline:    0.06408316 +/- 0.001840 (2.87%) (init= 0.03)
        ceiling:     1 (fixed)
        floor:       0 (fixed)
    [[Correlations]] (unreported correlations are <  0.100)
        C(amplitude, sigma)          = -0.491 
        C(sigma, baseline)           = -0.368 
        C(amplitude, baseline)       = -0.185 


Null model::

    [[Model]]
        Model(constant)
    [[Fit Statistics]]
        # function evals   = 11
        # data points      = 555
        # variables        = 1
        chi-square         = 1.282
        reduced chi-square = 0.002
        Akaike info crit   = -3367.048
        Bayesian info crit = -3362.729
    [[Variables]]
        c:   0.08062545 +/- 0.002042 (2.53%) (init= 0.03)



Comments
--------


.. raw:: html

    <div id="disqus_thread"></div>
    <script>
    
    (function() { // DON'T EDIT BELOW THIS LINE
    var d = document, s = d.createElement('script');
    s.src = 'https://agam-selection-atlas.disqus.com/embed.js';
    s.setAttribute('data-timestamp', +new Date());
    (d.head || d.body).appendChild(s);
    })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments.</a></noscript>


